content(cms): create Post "tei-header/index-1" (#26)
acdh-oeaw/howto: content/posts/tei-header/index-1.mdx

---
title: 4. TEI Header
lang: de
date: 2021-07-14T14:38:49.988Z
version: 1.0.0
authors:
  - czeitschner-ulrike
  - schopper-daniel
editors:
  - königshofer-elisabeth
  - wiencek-florian
tags:
  - tei
  - xml
abstract: Dieses Kapitel führt in grundlegende Elemente des TEI Headers ein, der
  Metadaten über ein TEI-Dokument erfasst.
licence: ccby-4.0
toc: false
uuid: j51QIatS7jhSnysibO4Cs
---

> ## Grundlagen

Reviewer: Bei diesem Kapitel sind keine Lernziele genannt.

Im Folgenden gehen wir auf die wichtigsten Metadaten-Informationen ein, die auch für unser Beispiel der "Filmwelt" relevant sind. In einer Übung werden dann die Informationen der zwei Seiten aus der "Filmwelt" entsprechend kodiert.

### Identifikation

Das **'title statement'** `<titleStmt>` gruppiert Informationen im Hinblick auf den Titel eines elektronischen Dokuments, sowie die Verantwortlichen für den Inhalt.

Hier ein Beispiel:

```xml
<titleStmt>
  <title type="main">Titel des elektronischen Dokuments</title>
  <title type="sub">Untertitel des elektronischen Dokuments</title>
  <respStmt>
    <resp>TEI Kodierung</resp>
    <persName xml:id="uc">Vorname Nachname</persName>
  </respStmt>
</titleStmt>
```

Das `<title>`-Element enhält den Titel des elektronischen Dokuments. Mit dem Attribut `@type` lassen sich unterschiedliche Arten von Titeln klassifizieren, beispielsweise:

* **'main'**: Haupt-Titel
* **'sub'**: Untertitel
* **'alt'**: Alternativer Titel unter dem das Werk ebenso bekannt ist, oft in einer anderen Sprache
* **'short'**: Kurzform des Titels

> * **'desc'**: deskriptive Paraphrasierung der Arbeit die als Titel fungiert.

Reviewer 2: Ja genau, solche Dinge wie "Ein Gedicht" oder auch "Arbeitstitel" - etwas, das das Werk näher beschreibt, indem es z.B. die Textsorte benennt.

```xml
<bibl>Gold, Matthew K., ed. 2012. Debates in the Digital Humanities. Minneapolis, London: University of Minnesota Press.</bibl>
```

Man kann diese Literaturangabe aber auch weiter auszeichnen, indem man die einzelne Teile der Literaturangabe semantisch benennt. Hierbei werden die Tags beispielsweise innerhalb der Satzstruktur der Literaturangabe verwendet.

```xml
<bibl><editor>Matthew K. Gold</editor>,ed., <date>2012</date>. <title>Debates in the Digital Humanities</title>. <pubPlace>Minneapolis</pubPlace>, <pubPlace>London</pubPlace>: <publisher>University of Minnesota Press</publisher>.</bibl>
```

Über das Element **'structured bibliographic citation'** `<biblStruct>` kann die Angabe noch weiter semantisch strukturiert werden.

Das obige Beispiel könnte in einer möglichen strukturierter Form wie folgt aussehen:

```xml
<biblStruct>
  <monogr>
    <editor>Matthew K. Gold</editor>
    <title>Debates in the Digital Humanities</title>
    <imprint>
      <publisher>University of Minnesota Press</publisher>
      <pubPlace>Minneapolis and London</pubPlace>
      <date>2012</date>
    </imprint>
  </monogr>
</biblStruct>
```

> Hierbei wird innerhalb der **'structured bibliographic citation'** ein Buch oder eine Zeitschrift mit dem Element **'monographic level'** `<monogr>` umschlossen und die Information so gruppiert. Innerhalb dessen können Autor*innen mit dem `<author>` sowie Editor*innen eines Buches oder einer Zeitschrift mit dem Element `<editor>` benannt werden.

Reviewer: Im CMS scheint das Gendersternchen so nicht auf. Vielleicht liegt es an der Formatierung bei Autor*innen und Editor*innen hier?

Reviewer 3: Habe es zur Zeit auf Autorinnen und Autoren geändert, Kursivschreibung kommt von der Verwendung des Sternchens im Markdown als Verweis auf kursiv. Das müssen wir nochmal anschauen; frage mich gerade, ob man im Deutschen statt Editor nicht Lektor sagen würde

`<imprint>` gruppiert vorhandene Informationen zur Publikation und Distribution des Werks.

> Der **'scope of bibliographic reference'** `<biblScope>`definiert, auf welchen Teil einer Arbeit sich die Referenz bezieht. Dies können z.B. bestimmte Seiten sein, die mit dem Attribut `unit="page"` angegeben werden, oder ein bestimmter Jahrgang einer Zeitschrift `unit="volume"`.

Reviewer: ```suggestion
Der **'scope of bibliographic reference'** `<biblScope>` definiert, auf welchen Teil einer Arbeit sich die Referenz bezieht. Dies können z.B. bestimmte Seiten sein, die mit dem Attribut `unit="page"` angegeben werden, oder ein bestimmter Jahrgang einer Zeitschrift, der mit `unit="volume"`ausgezeichnet wird.
```

      validate="input"
    >
      <Quiz.Question>
        **Bibliographische Informationen:** Die Filmwelt Illustrierte Kino Revue Herausgeber Felix Brasch Zeitungsverlag Universale Ges. m. b. H. Wien, VII., Hermanngasse 5 Druckerei L. Beck und Sohn Nr. 31, 1924 Seite 11-12

        **Verwenden Sie hierfür folgende Elemente / Attribute / Werte:**

        1. Geben Sie Haupt- und Untertitel der Zeitschrift an `<title>` + @type main / sub
        2. Benennen Sie den Herausgeber der Zeitschrift `<editor>`
        3. Ergänzen Sie im Impressum den Verlag inkl. seiner Adresse im Element `<imprint>`. Der **Verlag** `<publisher>` enthält hierbei den **Namen der Firma / Organisation** `<orgName>`, samt ihrer **Adresse** `<address>` bestehend aus einer oder mehrerer **Adresszeilen** `<addrLine>`.
        4. Weisen Sie die Fa. 'L. Beck und Sohn' in einem 'responsibility statement' als Druckerei aus unter Verwendung von `<resp>`, `<orgName>`
        5. Geben Sie Wien als Publikationsort und das Erscheinungsjahr an `<pubPlace>`, `<date>`

>         6. Kodieren Sie, dass der gewählte Textausschnitt aus der Nr. 31, 1924, Seite 11-12 stammt 'scope of bibliographic reference' `<biblScope>` + @type volume / page

Reviewer: ```suggestion
        6. Kodieren Sie, dass der gewählte Textausschnitt aus der Nr. 31, 1924, Seite 11-12 stammt mit Hilfe von 'scope of bibliographic reference' `<biblScope>` + @type volume / page
```

      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>
</Quiz>

### Kodierung

> Der bei dieser Übung entstandene TEI-Header umfasst nur einige wesentliche Elemente, er kann im Bedarfsfall weitaus detaillierter ausfallen. Ergänzend kann dieser beispielhaft auch einige Angaben zur Annotation **'encoding description'** `<encodingDesc>` enthalten, die den Zusammenhang der physischen Ressource und des elektronischen Dokuments beschreibt. Diese reichen von Informationen zu Redaktionsprinzipien über Projektbeschreibungen, Angaben zur Verwendung von Tags bis hin zur  zugrundeliegenden Textauswahl.

Reviewer: ```suggestion
Der bei dieser Übung entstandene TEI-Header umfasst nur einige wesentliche Elemente, er kann im Bedarfsfall weitaus detaillierter ausfallen. Ergänzend kann dieser beispielhaft auch einige Angaben zur Annotation **'encoding description'** `<encodingDesc>` enthalten, die den Zusammenhang der physischen Ressource und des elektronischen Dokuments beschreibt. Diese reichen von Informationen zu Redaktionsprinzipien über Projektbeschreibungen und Angaben zur Verwendung von Tags bis hin zur zugrundeliegenden Textauswahl.
```

```xml
<encodingDesc>
  <editorialDecl>
    <p>Die Annotation folgt den Richtlinien der TEI/P5.</p>
    <hyphenation>
      <p>Zeilenenden sind mit dem Element 'lb' markiert. Trennungsstriche
        wurden entfernt und sind durch das Attribut @break mit dem Wert
        'no' repräsentiert.</p>
    </hyphenation>
    <normalization>
      <p>Typographie und Orthographie wurden weitestgehend beibehalten.</p>
    </normalization>
    <correction>
      <p>Vorgenommene Korrekturen wurden als solche annotiert.</p>
    </correction>
    <interpretation>
      <p>Personennamen historischer, religiöser, fiktionaler Figuren,
        Ortsnamen, Organisationsbezeichnungen, Datumsangaben</p>
    </interpretation>
  </editorialDecl>
</encodingDesc>
```

> Hier einige Beispiele für mögliche Angaben innerhalb der 'encoding description':

Reviewer: ```suggestion
Hier einige Beispiele für mögliche Angaben innerhalb der **'encoding description'**:
```

> * **editorische Angaben:** beschreibt die angewandten Richtlinien und Prinzipien für die Annotation `<editorialDecl>`.

Reviewer: ```suggestion
* **editorische Angaben:** beschreiben die angewandten Richtlinien und Prinzipien für die Annotation `<editorialDecl>`
```

>   Wie in obenstehendem Beispiel können u.a. der Umgang mit Silbentrennung `<hyphenation>`, Normalisierung und Regulierung der originalen Quelle `<normalization>`, Korrekturen `<correction>` oder das Ausmaß an analytischer und interpretativer Information, die dem Text in der Transkription hinzugefügt wurde `<interpretation>`.

Reviewer: ```suggestion
  Wie in obenstehendem Beispiel können u.a. der Umgang mit Silbentrennung `<hyphenation>`, Normalisierung und Regulierung der originalen Quelle `<normalization>`, Korrekturen `<correction>` oder das Ausmaß an analytischer und interpretativer Information, die dem Text in der Transkription hinzugefügt wurde `<interpretation>`, angegeben werden
```

> * **Projektbeschreibung:** beschreibt das Ziel für das die Quelle kodiert wurde `<projectDesc>`

Reviewer: Ziel - etwas unklar; eventuell: Forschungsprojekt/Forschungsfrage?

> * **Tagverwendung:** gibt informationen zum Tagging, das innerhalb des Dokuments angewendet wird `<tagsDecl>`

Reviewer: ```suggestion
* **Tagverwendung:** gibt Informationen zum Tagging, das innerhalb des Dokuments angewendet wird `<tagsDecl>`
```
Hier könnte man auch "Tags" näher erklären, falls es vorher noch nicht passiert ist; oder auf ein Glossar verlinken.

Reviewer 3: guter Punkt

* **Auswahlverfahren:** Fließtext, der das angewandte Samplingverfahren für einen Korpus oder eine Sammlung an Texten beschreibt `<samplingDecl>`

#### Übung

<Quiz>
  <Quiz.Card>
    <Quiz.MultipleChoice>
      <Quiz.Question>
        Welche Aspekte eines Dokuments werden in der Encoding Description eines Dokuments erfasst? (mehrere Antworten möglich)
      </Quiz.Question>

      <Quiz.MultipleChoice.Option isCorrect>
        editorische Angaben
      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>

>         Informationen über Glyphen und Beschreibung von nicht-standard Zeichenformaten, die im Quelltext verwendet werden

Reviewer: Schreibweise: nicht-standard oder nicht-Standard
habe beides schon öfters gesehen - sollte einheitlich sein

```xml
<publicationStmt>
  <publisher>
    <orgName>Austrian Centre for Digital Humanities</orgName>
    <email>acdh@oeaw.ac.at</email>
    <idno type="VIAF">6515148451584915970000</idno>
  </publisher>
  <pubPlace>Vienna</pubPlace>
  <date>2017</date>
  <availability>
    <licence target="https://creativecommons.org/licenses/by-sa/4.0/">
      <p>Veröffentlicht unter CC-BY-SA 4.0</p>
    </licence>
  </availability>
</publicationStmt>
```

Wie am ersten Beispiel ersichtlich reicht ein einfacher Absatz um eine Publikation zu beschreiben, oder aber man verwendet eine deutlich komplexere semantische Schreibweise. Einige Elemente wie `<publisher>`, `<orgName>`, `<pubPlace>` oder `<date>` im obigen Beispiel sind bereits bekannt.

Das Element **'electronic mail address'** enthält eine Emailadresse. Das Element **'identifier'** `<idno>` enthält eine Form der standardisierten Identifikation eines Objekts -- in unserem Fall eines bibliografischen Objekts. Das Attribut `type` gibt die Art der Identifikation an. Diese kann beispielsweise folgende sein:

* **ISBN**: "International Standard Book Number" (ISBN-Nummer), die Bücher international eindeutig identifiziert.

> * **ISSN:** "International Standard Serial Number" (ISSN-Nummer), eine international einheitliche Indentifikationsnummer für serielle Publikationen, wie z.B. Zeitschriften.

Reviewer: ```suggestion
* **ISSN:** "International Standard Serial Number" (ISSN-Nummer), eine international einheitliche Identifikationsnummer für serielle Publikationen, wie z.B. Zeitschriften.
```

* **DOI:** "Digital Object Identifier", eine einmalige Zeichenfolge (aus Nummern und Buchstaben) die ein elektronisches Dokument identifiziert.

> * **URI:** Uniform Resource Identifier, eine eindeutige Identifikation einer Ressource (von physischen Objekte bis hin zu digitalen Ressourcen wie Webseiten), die gleichzeitig Hinweise auf den Zugriff der Ressource gibt.

Reviewer: ```suggestion
* **URI:** Uniform Resource Identifier, eine eindeutige Identifikation einer Ressource (von physischen Objekten bis hin zu digitalen Ressourcen wie Webseiten), die gleichzeitig Hinweise auf den Zugriff der Ressource gibt.
```

> * **VIAF:** Eine Normdatennummer im Virtual International Authority File (VIAF), einer virtuellen internationalen Normdatei als Verknüpfung nationaler Normdateien z.B. der gemeinsamen Normdatei. Diese wird als Uniform Resource Identifier für Linked-Open-Data Anwendungen genutzt.

Reviewer: ```suggestion
* **VIAF:** Eine Normdatennummer im Virtual International Authority File (VIAF), einer virtuellen internationalen Normdatei als Verknüpfung nationaler Normdateien. Diese wird als Uniform Resource Identifier für Linked-Open-Data Anwendungen genutzt.
```
Braucht man  hier den Teil  mit z.B. noch?

Das Element `<availability>` gibt Informationen zur Verfügbarkeit des Textes sowie Einschränkungen zur Nutzbarkeit, der Verbreitung, dem Copyright Status, oder der Lizenzierung. Die **Lizenzinformation** wird innerhalb des **availability-Elements** mittels des Elements `<licence>` angegeben. Das Attribut `target` wird hierbei zur Referenzierung des Volltextes der Lizenz verwendet.

#### Übung

<Quiz>
  <Quiz.Card>
    <Quiz.MultipleChoice>
      <Quiz.Question>
        Was wird im Publication Statement dokumentiert?
      </Quiz.Question>

      <Quiz.MultipleChoice.Option>
        Das Publication Statement enhält die Angaben zur Publikation des originalen Textes in Print.
      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>

>         Das Publication Statement enhält alle Informationen rund um die Publikation und Distribution des elektronischen Texts.

Reviewer: ```suggestion
        Das Publication Statement enthält alle Informationen rund um die Publikation und Distribution des elektronischen Texts.
```

      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option>

>         Das Publication Statement enhält das Vorwort des Herausgebers zur Publikation.

Reviewer: ```suggestion
        Das Publication Statement enthält das Vorwort des Herausgebers zur Publikation.
```

      validate="input"
    >
      <Quiz.Question>
        Fügen Sie dem bisherigen Dokument den ersten Eintrag der Revision History mit dem Datum 21.04.2021 und dem Namenskürzel von Maxi Musterfrau (mm) als Autorin hinzu.

>         Markieren Sie das Dokument als Entwurf und beschreiben Sie den Status als "Erster Entwurf abgeschlossen."

Reviewer: eventuell hier auch angeben, was für "Entwurf" (draft) verwendet werden soll
Zeichensetzung am Ende: "Erster Entwurf abgeschlossen".

      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>
</Quiz>

### Inhalt & Kontext

> Auch Informationen zum Inhalt des elektronischen Dokuments und seinem Kontext können bereits im Header erfasst werden. Hierfür kommt die **'text-profile description'** `<profileDesc>` zum Einsatz, die nicht-bibliographische Aspekte des Textes wie Sprachen, Produktionskontext oder die vorkommenden Personen / Stimmen erfasst.

Reviewer: ```suggestion
Auch Informationen zum Inhalt des elektronischen Dokuments und seinem Kontext können bereits im Header erfasst werden. Hierfür kommt die **'text-profile description'** `<profileDesc>` zum Einsatz, die nicht-bibliographische Aspekte des Textes wie Sprachen, Produktionskontext oder die vorkommenden Personen und Stimmen erfasst.
```

---

> title: 4. TEI Header

Reviewer 2: ```suggestion
title: 4. Der TEI-Header
```

> title: 4. TEI Header

Reviewer 2: ```suggestion
title: 4. TEI-Header
```

lang: de
date: 2021-07-14T14:38:49.988Z
version: 1.0.0
authors:
  - czeitschner-ulrike
  - schopper-daniel
editors:
  - königshofer-elisabeth
  - wiencek-florian
tags:
  - tei
  - xml

> abstract: Dieses Kapitel führt in grundlegende Elemente des TEI Headers ein, der

Reviewer 2: ```suggestion
abstract: Dieses Kapitel stellt den TEI-Header und seine Komponenten vor, die dazu dienen, einen kodierten Text detailliert zu beschreiben.
```

>   Metadaten über ein TEI-Dokument erfasst.

Reviewer 2: ```suggestion
```
=> Mit Satz darüber zusammengelegt.

licence: ccby-4.0
toc: false
uuid: j51QIatS7jhSnysibO4Cs
---

> ## Lernziele

Reviewer 2: ```suggestion
## Lerninhalte
```

> - Angabe der Metadaten eines TEI-Dokuments

Reviewer 2: ```suggestion
-  Metadaten für TEI-Dokumente
- Komponenten und Struktur
```

> - Strukturierung des Headers

Reviewer 2: ```suggestion
- Komponenten und Struktur
```

Author: Das "Ihre" bezieht sich wahrscheinlich auf die Metadaten im Bulletpoint davor, allerdings würde ich hier der Klarheit halber und zur Abkapselung der einzelnen Bulletpoints hier entweder die Metadaten oder den Header wiederholen.

Reviewer 2: Ja, da hast du völlig recht - danke!
Bitte stattdessen:
Komponenten und Strukturierung des TEI-Header

Author: ```suggestion
- Komponenten und Strukturierung des TEI-Header
```

## Grundlagen

> Im TEI-Header werden die Metadaten *über* ein TEI-Dokument erfasst. Der Header ist vom Element `<teiHeader>` umschlossen, die Metadaten über das Dokument selber werden innerhalb des Elements `<fileDesc>` angegeben, was das erste Element innerhalb des Headers darstellt. Diese **'file description'** enthält alle Informationen zum elektronischen Dokument (Name des elektronischen Dokuments, Autor*in etc.).

Reviewer 2: ```suggestion
Im `<teiHeader>`, der als oberstes Element innerhalb des Wurzelelements `<TEI>` positioniert ist, werden die Metadaten _über_ ein TEI-Dokument erfasst. Für die detaillierte Beschreibung der unterschiedlichen Aspekte steht eine Vielzahl an Komponenten zur Verfügung.
```

> Ein TEI-Header ist für jedes TEI-Dokument obligatorisch. Er kann nur einige wesentliche Elemente enthalten, kann aber auch sehr detailliert sein, wenn es nötig ist. Die Metadaten können u.a. folgende Informationen enthalten:

Reviewer 2: ```suggestion
Der TEI-Header ist für jedes TEI-Dokument obligatorisch. Er kann nur einige wesentliche Elemente enthalten, bei Bedarf jedoch sehr detailreich sein. U.a. sind in diesem Zusammenhang folgende Informationen relevant:
```

Author: Sehr klar und verständlich. Lediglich bei xenoData wäre vielleicht ein Beispiel hilfreich (was sind non-TEI metadata?).

Reviewer 2: Auch völlig richtig! Bitte ersetzen durch:
nimmt Metadaten in nicht-TEI-Formaten wie z.B. [Dublin Core](https://dublincore.org/) auf.

> 

Reviewer 2: ```suggestion

Um den Rahmen dieses Curriculums nicht zu sprengen, werden im Folgenden nur die wichtigsten Elemente vorgestellt: 
```

> 

Reviewer 2: ```suggestion

Um den Rahmen dieses Curriculums nicht zu sprengen, werden im weiteren Verlauf dieses Abschnitts nur einige ausgewählte Elemente näher vorgestellt. 
```

> - **Identifikation** ("Was ist das?") `<titleStmt>`

Reviewer 2: ```suggestion
- **Identifikation** _"Was ist das?"_ `<titleStmt>` (title statement)
```

> - **Identifikation** ("Was ist das?") `<titleStmt>`

Reviewer 2: ```suggestion
- **Identifikation** _"Was ist das?"_ `<titleStmt>` (title statement)
```

> - **Identifikation** ("Was ist das?") `<titleStmt>`

Reviewer 2: ```suggestion
- **Identifikation** _"Was ist das?"_ `<titleStmt>` (Teil der "file description")
```

> - **Identifikation** ("Was ist das?") `<titleStmt>`

Reviewer 2: ```suggestion
```
=> Umreihung

> - **Angabe der Quelle** ("Woraus wurde es abgeleitet?") `<sourceDesc>`

Reviewer 2: ```suggestion
- **Angabe der Quelle** _"Woraus wurde es abgeleitet?"_ `<sourceDesc>` (Teil der "file description")
```

> - **Kodierung** ("Wie wurde der Text kodiert?") `<encodingDesc>`

Reviewer 2: ```suggestion
- **Kodierung** _"Wie wurde der Text kodiert?"_ `<encodingDesc>`
```

> - **Kodierung** ("Wie wurde der Text kodiert?") `<encodingDesc>`

Reviewer 2: ```suggestion
- **Kodierung** _"Wie wurde der Text kodiert?"_ `<encodingDesc>`
```

> - **Kodierung** ("Wie wurde der Text kodiert?") `<encodingDesc>`

Reviewer 2: ```suggestion
```
=> Umreihung

> - **Veröffentlichung** ("Wer gibt es unter welchen Bedingungen für wen frei?") `<publicationStmt>`

Reviewer 2: ```suggestion
- **Veröffentlichung** ("Wer gibt das Dokument unter welchen Bedingungen für wen frei?") `<publicationStmt>`
```

> - **Veröffentlichung** ("Wer gibt es unter welchen Bedingungen für wen frei?") `<publicationStmt>`

Reviewer 2: ```suggestion
```
=> Umreihung

> - **Provenienz** ("Wer hat was wann damit gemacht?") `<revisionDesc>`

Reviewer 2: ```suggestion
- **Provenienz** _"Wer hat was wann damit gemacht?"_ `<revisionDesc>`
```

> - **Provenienz** ("Wer hat was wann damit gemacht?") `<revisionDesc>`

Reviewer 2: ```suggestion
```
=> Umreihung

## Praktische Anwendung

> Im Folgenden ist eine einfache Grundstruktur eines TEI Dokuments abgebildet, auf die wir in unseren Übungen aufbauen werden.

Reviewer 2: ```suggestion
Als Ausgangspunkt für die weiteren Ausführungen dient folgende Minimalversion eines validen TEI-Dokuments:
```

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/tei_all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/tei_all.rng" type="application/xml"
	schematypens="http://purl.oclc.org/dsdl/schematron"?>
<TEI xmlns="http://www.tei-c.org/ns/1.0">
  <teiHeader>
    <fileDesc>
      <titleStmt>

>         <title>Title</title>

Reviewer 2: ```suggestion
        <title>Titel</title>
```

      </titleStmt>
      <publicationStmt>

>         <p>Publication Information</p>

Reviewer 2: ```suggestion
        <p>Informationen zur Veröffentlichung</p>
```

      </publicationStmt>
      <sourceDesc>

>         <p>Information about the source</p>

Reviewer 2: ```suggestion
        <p>Informationen über die Quelle</p>
```

      </sourceDesc>
    </fileDesc>
  </teiHeader>
  <text>
    <body>

>       <p>Some text here.</p>

Reviewer 2: ```suggestion
      <p>Hier ein Text.</p>
```

    </body>
  </text>
</TEI>
```

> Der TEI-Header besteht in unserem Fall aus der **'file description'**, die alle Informationen zum elektronischen Dokument enthält (Name des elektronischen Dokuments, Autor*in etc.). Nachdem wir in unserem Beispiel den Text nicht selbst verfassen, sondern einen Ausschnitt aus einer historischen Zeitschrift kodieren, ist darin auch eine Quellenangabe **'source description'** `<sourceDesc>` enthalten.

Reviewer 2: ```suggestion
Innerhalb der **'file description'** ist neben dem **'title statement'**, das den Titel des TEI-Dokuments und die beteiligten Autor*innen angibt, eine Quellenbeschreibung **'source description'** obligatorisch.
```
(=> Kein Beistrich vor "etc." ...)

> Der TEI-Header besteht in unserem Fall aus der **'file description'**, die alle Informationen zum elektronischen Dokument enthält (Name des elektronischen Dokuments, Autor*in etc.). Nachdem wir in unserem Beispiel den Text nicht selbst verfassen, sondern einen Ausschnitt aus einer historischen Zeitschrift kodieren, ist darin auch eine Quellenangabe **'source description'** `<sourceDesc>` enthalten.

Reviewer 2: ```suggestion
Da die Metadaten im TEI-Header so ausführlich wie möglich sein sollen, empfiehlt es sich, neben den hier angegebenen Pflichtangaben auch einige der zur Verfügung stehenden optionalen Elemente zu verwenden. Als Vorlage für die konkreten Übungen dient der in Kapitel 2 vorgestellte Beispieltext _Die Filmwelt_. Die für die Befüllung der Elemente notwendigen Informationen sind bei der jeweiligen Übung angegeben.
```
=> "Kapitel 2" verlinken

> Im Folgenden gehen wir auf die wichtigsten Metadaten-Informationen ein, die auch für unser Beispiel der *Filmwelt* relevant sind. In einer Übung werden dann die Informationen der zwei Seiten aus der *Filmwelt* entsprechend kodiert.

Reviewer 2: ```suggestion
```
=> Passt hier nicht.

> Im Folgenden gehen wir auf die wichtigsten Metadaten-Informationen ein, die auch für unser Beispiel der *Filmwelt* relevant sind. In einer Übung werden dann die Informationen der zwei Seiten aus der *Filmwelt* entsprechend kodiert.

Reviewer 2: ```suggestion
```
=> Umformuliert, mit Absatz darüber zusammengelegt.

### Identifikation

> Das **'title statement'** `<titleStmt>` gruppiert Informationen im Hinblick auf den Titel eines elektronischen Dokuments, sowie die Verantwortlichen für den Inhalt.

Reviewer 2: ```suggestion
Im obligatorischen **'title statement'** `<titleStmt>` stehen für den Titel eines TEI-Dokuments das ebenso **obligatorische** `<title>`-Element und **optional** u.a. die Elemente `<author>` und `<editor>` für Autor*innen** und Herausgeber*innen eines Textes sowie das 'statement of responsibility'** `<respStmt>` zur Verfügung. Letzteres wird verwendet, wenn die spezifischen Elemente `<author>` und `<editor>` nicht ausreichen oder zutreffen. Das ist z.B. bei historischen retro-digitalisierten Texten der Fall, da die Autor*innen bzw. Herausgeber*innen nicht ident mit jenen Personen sind, die das elektronische Dokument erstellt haben. 
```

> 

Reviewer 2: ```suggestion
```

> Hier ein Beispiel:

Reviewer 2: ```suggestion
```

```xml
<titleStmt>
  <title type="main">Titel des elektronischen Dokuments</title>
  <title type="sub">Untertitel des elektronischen Dokuments</title>
  <respStmt>

>     <resp>TEI Kodierung</resp>

Reviewer 2: ```suggestion
    <resp>TEI-Kodierung</resp>
```

>     <persName xml:id="uc">Vorname Nachname</persName>

Reviewer 2: ```suggestion
    <persName>Vorname Nachname</persName>
```

  </respStmt>
</titleStmt>
```

> Das `<title>`-Element enhält den Titel des elektronischen Dokuments. Mit dem Attribut `@type` lassen sich unterschiedliche Arten von Titeln klassifizieren, beispielsweise:

Reviewer 2: ```suggestion
Im `<title>`-Element lassen sich mit Hilfe des Attributs `@type` unterschiedliche Arten von Titeln klassifizieren:
```

> - **'main'**: Haupt-Titel

Reviewer 2: ```suggestion
- **'main'**: Haupttitel
```

> - **'sub'**: Untertitel

Reviewer 2: ```suggestion
- **'sub'**: Untertitel bzw. Zwischentitel eines untergeordneten Abschnitts
```

> - **'alt'**: Alternativer Titel unter dem das Werk ebenso bekannt ist, oft in einer anderen Sprache

Reviewer 2: ```suggestion
- **'alt'**: Alternativer Titel
```
=> Ich will auch bei Defs. nicht ganz wortwörtlich von der TEI abschreiben ... ist auch so aussagekräftig genug.

> - **'short'**: Kurzform des Titels

Reviewer 2: ```suggestion
- **'short'**: Kurztitel
```

> - **'desc'**: deskriptive Paraphrasierung der Arbeit die als Titel fungiert.

Reviewer 2: ```suggestion
- **'desc'**: Titel in Form einer beschreibenden Paraphrase
```

> In der Praxis ist eine Klassifizierung der einzelnen Titelteile nicht unbedingt notwendig. Der komplette Titel mit Untertitel könnte auch in einem einzigen `<title>` Element geschrieben werden.

Reviewer 2: ```suggestion
In der Praxis ist eine Klassifizierung der einzelnen Titelteile nicht unbedingt notwendig. Der komplette Titel mit Untertitel könnte auch als ein `<title>`-Element annotiert werden. Dies könnte allerdings für die Erstellung von übersichtlichen Inhaltsverzeichnissen von Nachteil sein.
```

Author: Ja, dies sollten wir in ein Merke umwandeln

> Ergänzend zum Titel bietet das **'responsibility statement'** `<respStmt>` die Möglichkeit, die Verantwortlichkeiten für den intellektuellen Inhalt eines Textes, einer Edition oder dergleichen anzugeben. In diesem Beispiel ist es die Tätigkeit der "TEI Kodierung". Die Verantwortlichkeit wird im Element **'responsibility'** `<resp>` innerhalb des Statements benannt, die Person im Element **'personal name'** `<persName>`.

Reviewer 2: ```suggestion
Ergänzend zum Titel bietet das **'responsibility statement'** `<respStmt>` die Möglichkeit, die Verantwortlichkeiten für den intellektuellen Inhalt eines Textes, einer Edition oder dergleichen anzugeben. In diesem Beispiel ist es die Tätigkeit der "TEI Kodierung". Die Verantwortlichkeit wird im Element **'responsibility'** `<resp>` innerhalb des Statements benannt, die Person im Element **'personal name'** `<persName>`.
```
=> Der Urheber einer digitalen Edition ist bei retro-digitalisierten Texten eben **NICHT** für den intellektuellen Inhalt, sprich den Text verantwortlich! Das ist und bleibt der Autor des Textes. Das `<respStmt>`wird verwendet um eben diese Ebenen klar auseinander zu halten.

> Ergänzend zum Titel bietet das **'responsibility statement'** `<respStmt>` die Möglichkeit, die Verantwortlichkeiten für den intellektuellen Inhalt eines Textes, einer Edition oder dergleichen anzugeben. In diesem Beispiel ist es die Tätigkeit der "TEI Kodierung". Die Verantwortlichkeit wird im Element **'responsibility'** `<resp>` innerhalb des Statements benannt, die Person im Element **'personal name'** `<persName>`.

Reviewer 2: ```suggestion
Ergänzend zum Titel bietet das **'responsibility statement'** `<respStmt>` die Möglichkeit, die Verantwortlichkeiten für den intellektuellen Inhalt eines Textes, einer Edition oder dergleichen anzugeben. In diesem Beispiel ist es die Tätigkeit der "TEI Kodierung". Die Verantwortlichkeit wird im Element **'responsibility'** `<resp>` innerhalb des Statements benannt, die Person im Element **'personal name'** `<persName>`.
```
=> Der Urheber einer digitalen Edition ist bei retro-digitalisierten Texten **NICHT** zwingend auch für den intellektuellen Inhalt verantwortlich, sprich der Verfasser des Textes! Das ist und bleibt der Autor des Textes. Da wird häufig die Verwendung von `<author>` vermieden und stattdessen `<respStmt>` verwendet um diese Ebenen klar auseinander zu halten.

> Ergänzend zum Titel bietet das **'responsibility statement'** `<respStmt>` die Möglichkeit, die Verantwortlichkeiten für den intellektuellen Inhalt eines Textes, einer Edition oder dergleichen anzugeben. In diesem Beispiel ist es die Tätigkeit der "TEI Kodierung". Die Verantwortlichkeit wird im Element **'responsibility'** `<resp>` innerhalb des Statements benannt, die Person im Element **'personal name'** `<persName>`.

Reviewer 2: ```suggestion
Neben dem Titel bietet das **'responsibility statement'** `<respStmt>` die **optionale** Möglichkeit, die Verantwortlichkeiten für den intellektuellen Inhalt eines Textes, einer Edition, einer Aufnahme oder dergleichen anzugeben. Es kommt zum Einsatz, wenn die spezifischen Elemente `<author>` und `<editor>` nicht ausreichen oder zutreffen. Mitunter ist es z.B. der Fall, dass die Verfasser*innen des im Dokument enthaltenen Textes nicht ident mit jenen Personen sind, die für die Erstellung des elektronischen Dokuments verantwortlich sind. Um hier Missverständnisse in Bezug auf die Urheberschaft zu vermeiden und mehr Spielraum zu haben, können im `<respStmt>` eindeutige Rollenzuweisungen für alle am Digitalisat Beteiligten, die weder `<author>` noch `<editor>` genannt werden können, vorgenommen werden. Die Mehrfachverwendung des Elements ist möglich um jeder involvierten Person oder Organisation ihre spezifische Verantwortung zuzuweisen.

Im oben angeführten Beispiel ist es die "TEI-Kodierung". Die Tätigkeit wird im Element **'responsibility'** `<resp>` genannt, die dafür verantwortliche Person im Element **'personal name'** `<persName>`. Handelte es sich um eine Organisation, käme das Element **'organization name'** `<orgName>` zum Einsatz. Wollte man darauf verzichten, Personen von Organisationen zu unterscheiden, könnte man beide mit dem weniger spezifischen Element **'name'** `<name>` annotieren. Wie schon erwähnt, gibt es in der TEI häufig mehrere Möglichkeiten mit jeweils unterschiedlichen Vor- und Nachteilen. Welche Wahl man trifft, hängt nicht zuletzt von den Zielen, die man erreichen möchte, ab.
```

> #### Übung

Reviewer 2: ```suggestion
#### Quiz
```

Author: Hier sprechen wir jetzt von Quiz, obwohl es nachher eine Übungsaufgabe und kein Multiple Choice o.ä. ist. Im Text steht weiterhin Übung. Hier sollten wir einheitlich mit der Anpassung sein. Ist es nun ein Quiz oder eine Übung?

Reviewer 2: Ja sorry stimmt, hier muss Übung bleiben.

> In der ersten Übung sollen die ersten Grundangaben zum elektronischen Dokument des Ausschnitts der Zeitschrift "Filmwelt" gemacht werden. Der Startpunkt ist das TEI Grundgerüst. Vervollständigen Sie dort bitte den TEI-Header um folgende Angaben:

Reviewer 2: ```suggestion
In dieser Übung sollen die ersten Grundangaben zum elektronischen Dokument des Ausschnitts der Zeitschrift _Die Filmwelt_ gemacht werden. Der Startpunkt ist das TEI Grundgerüst. Vervollständigen Sie dort bitte den TEI-Header um folgende Angaben:
```
=> Die Zeitschrift heißt _Die Filmwelt_!
**=> ... in progress ...**

> In der ersten Übung sollen die ersten Grundangaben zum elektronischen Dokument des Ausschnitts der Zeitschrift "Filmwelt" gemacht werden. Der Startpunkt ist das TEI Grundgerüst. Vervollständigen Sie dort bitte den TEI-Header um folgende Angaben:

Reviewer 2: ```suggestion
In dieser Übung sollen erste Metadaten für die digitale Version des Beispieltextes _Die Filmwelt_ annotiert werden. Bitte vervollständigen Sie im vorgegebenen TEI-Grundgerüst den TEI-Header um folgende Angaben:
```
=> Die Zeitschrift heißt _Die Filmwelt_!
**=> ... in progress ...**

<Quiz>
  <Quiz.Card>
    <Quiz.XmlCodeEditor

>       code={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title>Title</title>\n      </titleStmt>\n      <publicationStmt>\n        <p>Publication Information</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Information about the source</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Some text here.</p>\n    </body>\n  </text>\n</TEI>"}

Reviewer 2: ```suggestion
      code={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title>Titel</title>\n      </titleStmt>\n      <publicationStmt>\n        <p>Information zur Veröffentlichung</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Information über die Quelle</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text.</p>\n    </body>\n  </text>\n</TEI>"}
```

>       solution={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt - ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n      </titleStmt>\n      <publicationStmt>\n        <p>Publication Information</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Information about the source</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Some text here.</p>\n    </body>\n  </text>\n</TEI>"}

Reviewer 2: ```suggestion
      solution={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt - ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version.</title>\n      </titleStmt>\n      <publicationStmt>\n        <p>Information zur Veröffentlichung</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Information über die Quelle</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text.</p>\n    </body>\n  </text>\n</TEI>"}
```

      validate="input"
    >
      <Quiz.Question>

>         1. Geben Sie einen vollständigen Titel für das TEI-Dokument an: ersetzen Sie das `<title>`-Element der Vorlage durch:

Reviewer 2: ```suggestion
        1. Geben Sie einen vollständigen Titel für das TEI-Dokument an. Ersetzen Sie das `<title>`-Element der Vorlage durch:
```

>         - **Titel des elektronische Dokuments:** Die Filmwelt - ein Ausschnitt.

Reviewer 2: ```suggestion
        - **Titel des elektronischen Dokuments:** Die Filmwelt - ein Ausschnitt.
```

>         - **Untertitel des elektronischen Dokuments:** Digitale Version.

Reviewer 2: ```suggestion
        - **Untertitel des elektronischen Dokuments:** Digitale Version
```

      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>

  <Quiz.Card>
    <Quiz.XmlCodeEditor

>       code={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt - ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n      </titleStmt>\n      <publicationStmt>\n        <p>Publication Information</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Information about the source</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Some text here.</p>\n    </body>\n  </text>\n</TEI>"}

Reviewer 2: ```suggestion
      code={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt - ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n      </titleStmt>\n      <publicationStmt>\n        <p>Information zur Publikation</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Information über die Quelle</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text</p>\n    </body>\n  </text>\n</TEI>"}
```

>       code={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt - ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n      </titleStmt>\n      <publicationStmt>\n        <p>Publication Information</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Information about the source</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Some text here.</p>\n    </body>\n  </text>\n</TEI>"}

Reviewer 2: ```suggestion
      code={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt - ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n      </titleStmt>\n      <publicationStmt>\n        <p>Informationen zur Publikation</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Informationen über die Quelle</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text.</p>\n    </body>\n  </text>\n</TEI>"}
```

      solution={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt - ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n        <respStmt>\n          <resp>TEI Kodierung</resp>\n          <persName>Maxi Musterfrau</persName>\n        </respStmt>\n      </titleStmt>\n      <publicationStmt>\n        <p>Publication Information</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Information about the source</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Some text here.</p>\n    </body>\n  </text>\n</TEI>"}
      validate="input"
    >
      <Quiz.Question>

>         2. Ergänzen Sie bitte im Element **'title statement'** das **'statement of responsibility'** mit

Reviewer 2: ```suggestion
        2. Ergänzen Sie bitte im Element **'title statement'** das **'statement of responsibility'** mit den Elementen:
```

>         den Elementen **'responsibility'** und **'personal name'** und geben Sie _Maxi Musterfrau_ als Urheber\*in der TEI-Kodierung an:

Reviewer 2: ```suggestion
        den Elementen **'responsibility'** und **'personal name'** und geben Sie _Erika Musterfrau_ als Urheber\*in der TEI-Kodierung an:
```
=> Die heißt echt Erika Mustermann oder noch weiblicher Musterfrau 
https://commons.wikimedia.org/wiki/Category:Mustermann/Musterfrau_(generic_names)?uselang=de

>         den Elementen **'responsibility'** und **'personal name'** und geben Sie _Maxi Musterfrau_ als Urheber\*in der TEI-Kodierung an:

Reviewer 2: ```suggestion
        den Elementen **'responsibility'** und **'personal name'** und geben Sie _Erika Musterfrau_ als Urheber\*in der TEI-Kodierung an:
```

>         den Elementen **'responsibility'** und **'personal name'** und geben Sie _Maxi Musterfrau_ als Urheber\*in der TEI-Kodierung an:

Reviewer 2: ```suggestion
        den Elementen **'responsibility'** und **'personal name'** und geben Sie _Erika Musterfrau_ als Urheber\*in der TEI-Kodierung an:
```

      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>
</Quiz>

> ### Angabe der Quelle

Reviewer 2: ```suggestion
### Quellenbeschreibung
```

> Die **'source description'** `<sourceDesc>` gibt an, ob es sich um ein born-digital Dokument handelt oder im Falle von retrodigitalisiertem Material enthält sie Informationen über die Quelle, auf der das Dokument basiert. In der Regel ist dies bei einem digitalisierten Text eine bibliographische Quelle, die in vollständig strukturierter Form beispielsweise mit dem Element **'structured bibliographic citation'** `<biblStruct>` angegeben werden kann, oder einfacher strukturiert mit dem Element **'bibliographic citation'** `<bibl>`. Für einen digital erstellten Text ohne eine physische Quelle kann man einen Absatz wie zum Beispiel "Digitales Originaldokument" innerhalb eines Absatzes in der 'source description' einfügen.

Reviewer 2: ```suggestion
Die **'source description'** `<sourceDesc>` gibt Auskunft darüber, ob es sich um ein 'born-digital'-Dokument handelt oder auf anderen Quellen basiert – seien diese analog oder digital macht hier keinen Unterschied. Eine Beschreibung der Quellen kann als einfaches bibliografisches Zitat im Element **'bibliographic citation'** `<bibl>` erfolgen oder in vollständig strukturierter Form in **'structured bibliographic citation'** `<biblStruct>` vorgenommen werden. Für einen digital erstellten Text ohne physischen Quellen genügt eine kurze Angabe, die in einem `<p>`-Element (paragraph) innerhalb der 'source description' einzufügen ist.
```

> 

Reviewer 2: ```suggestion
```

> Hier ein Beispiel für ein "born digital" Dokument:

Reviewer 2: ```suggestion
```

```xml
<sourceDesc>

>   <p>Digitales Originaldokument</p>

Reviewer 2: ```suggestion
  <p>Digitales Originaldokument.</p>
```

</sourceDesc>
```

> Da wir es in unserem Fallbeispiel allerdings mit einer physischen Quelle zu tun haben, müssen wir in diesem Fall die Quelle nennen. Dies kann auf unterschiedliche Weisen geschehen.

Reviewer 2: ```suggestion
Da es sich im Falle unseres Beispieltextes _Die Filmwelt_ um einen digitalisierten Ausschnitt aus einer historischen Zeitschrift handelt, ist die physische Quelle zu nennen.
```

> Die minimalste Form ist die Literaturangabe in einem Zitationsformat innerhalb des Elements **'bibliographic citation'**.

Reviewer 2: ```suggestion
Die minimalste Form ist ein bibliografisches Zitat im Element **'bibliographic citation'**.
```

```xml
<bibl>Gold, Matthew K., ed. 2012. Debates in the Digital Humanities. Minneapolis, London: University of Minnesota Press.</bibl>
```

> Man kann diese Literaturangabe aber auch weiter auszeichnen, indem man die einzelne Teile der Literaturangabe semantisch benennt. Hierbei werden die Tags beispielsweise innerhalb der Satzstruktur der Literaturangabe verwendet.

Reviewer 2: ```suggestion
Darüber hinaus könnte man in dieser Literaturangabe einzelne Komponenten semantisch benennen, indem man weitere Tags innerhalb der Satzstruktur einfügt.
```

```xml

> <bibl><editor>Matthew K. Gold</editor>,ed., <date>2012</date>. <title>Debates in the Digital Humanities</title>. <pubPlace>Minneapolis</pubPlace>, <pubPlace>London</pubPlace>: <publisher>University of Minnesota Press</publisher>.</bibl>

Reviewer 2: ```suggestion
<bibl><editor>Matthew K. Gold</editor>, ed., <date>2012</date>. <title>Debates in the Digital Humanities</title>. <pubPlace>Minneapolis</pubPlace>, <pubPlace>London</pubPlace>: <publisher>University of Minnesota Press</publisher>.</bibl>
```

```

> 

Reviewer 2: ```suggestion

Diese Variante wird z.B. für die Annotation von Bibliografien mit vielen Literaturzitaten verwendet, deren "Satzstruktur" der Vorlage entsprechend abgebildet werden soll.
```

> Über das Element **'structured bibliographic citation'** `<biblStruct>` kann die Angabe noch weiter semantisch strukturiert werden.

Reviewer 2: ```suggestion
Für eine wirklich vollständige Strukturierung aller Teile ist jedoch nicht das `<bibl>`-Element, sondern **'structured bibliographic citation'** `<biblStruct>` vorgesehen, dessen Unterelemente nicht der Satzstruktur folgen, sondern in einer bestimmten Reihenfolge angeordnet sind.
```

> 

Reviewer 2: ```suggestion
```

> Das obige Beispiel könnte in einer möglichen strukturierter Form wie folgt aussehen:

Reviewer 2: ```suggestion
```

```xml
<biblStruct>
  <monogr>
    <editor>Matthew K. Gold</editor>
    <title>Debates in the Digital Humanities</title>
    <imprint>
      <publisher>University of Minnesota Press</publisher>
      <pubPlace>Minneapolis and London</pubPlace>
      <date>2012</date>
    </imprint>
  </monogr>
</biblStruct>
```

> Hierbei wird innerhalb der **'structured bibliographic citation'** ein Buch oder eine Zeitschrift mit dem Element **'monographic level'** `<monogr>` umschlossen und die Information so gruppiert. Innerhalb dessen können Autorinnen und Autoren *mit dem `<author>` sowie Editorinnen und Editoren* eines Buches oder einer Zeitschrift mit dem Element `<editor>` benannt werden.

Reviewer 2: ```suggestion
Im `<biblStruct>`-Element gruppiert das Element **'monographic level'** `<monogr>` alle bibliografischen Angaben eines Buches oder einer Zeitschrift. Autor\*innen und Editor\*innen werden mit den Elementen `<author>` bzw. `<editor>` annotiert. Dabei könnten zusätzlich die Elemente `<persName>`, `<forename>`, `<surname>` etc. zum Einsatz kommen. Letztere wurden hier nicht verwendet um die Übersichtlichkeit des Beispiels zu bewahren. Es folgt der Titel und daran anschließend das `<imprint>`-Element, das Informationen zur Veröffentlichung und Verbreitung zusammenfasst.
```

Reviewer 2: => Bitte auch diesen Text übernehmen:

> 

Reviewer 2: ```suggestion
```

> `<imprint>` gruppiert vorhandene Informationen zur Publikation und Distribution des Werks.

Reviewer 2: ```suggestion
```
=> Mit Absatz darüber zusammengelegt.

> Der **'scope of bibliographic reference'** `<biblScope>`definiert, auf welchen Teil einer Arbeit sich die Referenz bezieht. Dies können z.B. bestimmte Seiten sein, die mit dem Attribut `unit="page"` angegeben werden, oder ein bestimmter Jahrgang einer Zeitschrift `unit="volume"`.

Reviewer 2: ```suggestion
Nach dem Impressum kann bei Bedarf mit **'scope of bibliographic reference'** `<biblScope>` angegeben werden, auf welchen Teil einer Arbeit sich die bibliografische Referenz bezieht. Im Attribut `@unit` können Seiten- oder Jahrgangsangaben mit den Werten `page` und `volume` spezifiziert werden.
```

#### Übung

> In dieser Übung wollen wir die Literaturangabe unserer Quelle in einer strukturierten Bibliographie darstellen. Ersetzen Sie bitte den Inhalt der **'source description'** im Quellcode durch eine detaillierte bibliographische Angabe mit Hilfe der Elemente **'structured bibliographic citation'** `<biblStruct>` und **'monographic level'** `<monogr>`. Im Folgenden finden Sie die bibliographische Information unseres Zeitschriften-Auszugs, der in dieser Übung zum zuvor bearbeiteten TEI-Code ergänzt werden soll.

Reviewer 2: ```suggestion
In dieser Übung soll die Literaturangabe unserer Quelle als strukturierte Bibliografie annotiert werden. Ersetzen Sie bitte den Inhalt der **'source description'** im Quellcode durch eine detaillierte bibliografische Angabe mit Hilfe der Elemente **'structured bibliographic citation'** `<biblStruct>` und **'monographic level'** `<monogr>`. Im Folgenden finden Sie die bibliografischen Informationen unseres Zeitschriften-Auszugs, der in dieser Übung zum zuvor bearbeiteten TEI-Code ergänzt werden soll.
```

<Quiz>
  <Quiz.Card>
    <Quiz.XmlCodeEditor

>       code={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt - ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n        <respStmt>\n          <resp>TEI Kodierung</resp>\n          <persName>Maxi Musterfrau</persName>\n        </respStmt>\n      </titleStmt>\n      <publicationStmt>\n        <p>Publication Information</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Information about the source</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Some text here.</p>\n    </body>\n  </text>\n</TEI>"}

Reviewer 2: ```suggestion
      code={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt – ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n        <respStmt>\n          <resp>TEI Kodierung</resp>\n          <persName>Erika Musterfrau</persName>\n        </respStmt>\n      </titleStmt>\n      <publicationStmt>\n        <p>Informationen zur Veröffentlichung</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Informationen über die Quelle</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text.</p>\n    </body>\n  </text>\n</TEI>"}
```

>       solution={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt - ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n        <respStmt>\n          <resp>TEI Kodierung</resp>\n          <persName>Maxi Musterfrau</persName>\n        </respStmt>\n      </titleStmt>\n      <publicationStmt>\n        <p>Publication Information</p>\n      </publicationStmt>\n      <sourceDesc>\n        <biblStruct>\n          <monogr>\n            <title type=\"main\">Die Filmwelt</title>\n            <title type=\"sub\">Illustrierte Kino Revue</title>\n            <editor>Felix Brasch</editor>\n            <imprint>\n              <publisher>\n                <orgName>Zeitungsverlag Universale Ges. m. b. H.</orgName>\n                <address>\n                  <addrLine>Wien, VII., Hermanngasse 5</addrLine>\n                </address>\n              </publisher>\n              <respStmt>\n                <resp>Druckerei</resp>\n                <orgName>L. Beck und Sohn</orgName>\n              </respStmt>\n              <pubPlace>Wien</pubPlace>\n              <date>1924</date>\n            </imprint>\n            <biblScope unit=\"volume\">31, 1924</biblScope>\n            <biblScope unit=\"page\">11-12</biblScope>\n          </monogr>\n        </biblStruct>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Some text here.</p>\n    </body>\n  </text>\n</TEI>"}

Reviewer 2: ```suggestion
      solution={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt – ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n        <respStmt>\n          <resp>TEI Kodierung</resp>\n          <persName>Erika Musterfrau</persName>\n        </respStmt>\n      </titleStmt>\n      <publicationStmt>\n        <p>Informationen zur Veröffentlichung</p>\n      </publicationStmt>\n      <sourceDesc>\n        <biblStruct>\n          <monogr>\n            <title type=\"main\">Die Filmwelt</title>\n            <title type=\"sub\">Illustrierte Kino Revue</title>\n            <editor>Felix Brasch</editor>\n            <imprint>\n              <publisher>\n                <orgName>Zeitungsverlag Universale Ges. m. b. H.</orgName>\n                <address>\n                  <addrLine>Wien, VII., Hermanngasse 5</addrLine>\n                </address>\n              </publisher>\n              <respStmt>\n                <resp>Druckerei</resp>\n                <orgName>L. Beck und Sohn</orgName>\n              </respStmt>\n              <pubPlace>Wien</pubPlace>\n              <date>1924</date>\n            </imprint>\n            <biblScope unit=\"volume\">31, 1924</biblScope>\n            <biblScope unit=\"page\">11-12</biblScope>\n          </monogr>\n        </biblStruct>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Some text here.</p>\n    </body>\n  </text>\n</TEI>"}
```

Author: Ich sehe dass Beck & Sohn hier wieder auf die Nutzung des "&" geändert wurde.  Leider macht genau das Probleme in der Validierung und damit dem Rendering des MDX, deswegen hatte ich es durchgehend geändert...

Reviewer 2: Ich habe extra nicht & sondern &_amp; verwendet, würde &_#x26; helfen? Das kaufmännische Und steht halt in der Zeitschrift und sollte so abgebildet werden ...

      validate="input"
    >
      <Quiz.Question>

>         **Bibliographische Informationen:** Die Filmwelt Illustrierte Kino Revue Herausgeber Felix Brasch Zeitungsverlag Universale Ges. m. b. H. Wien, VII., Hermanngasse 5 Druckerei L. Beck und Sohn Nr. 31, 1924 Seite 11-12

Reviewer 2: ```suggestion
        **Bibliografische Informationen:** Die Filmwelt | Illustrierte Kino Revue | Herausgeber Felix Brasch | Zeitungsverlag Universale Ges. m. b. H. | Wien, VII., Hermanngasse 5 | Druckerei L. Beck & Sohn | Nr. 31, 1924 | Seite 11-12
```

Author: Die Verwendung des Trennstriches finde ich gut. Bezüglich "&" siehe Kommentar oben.

Reviewer 2: Hier habe tatsächlich nur & verwendet - ich dachte im Fließtext ist das egal. Das müsste aber schon möglich sein, das wäre ja unangenehm wenn wir das nicht hinbekämen ...

>         **Verwenden Sie hierfür folgende Elemente / Attribute / Werte:**

Reviewer 2: ```suggestion
        **Zu verwendende Elemente / Attribute / Werte:**
```

>         1. Geben Sie Haupt- und Untertitel der Zeitschrift an `<title>` + @type main / sub

Reviewer 2: ```suggestion
        1. Geben Sie Haupt- und Untertitel der Zeitschrift an `<title>` + `@type=main/sub`
```

>         2. Benennen Sie den Herausgeber der Zeitschrift `<editor>`

Reviewer 2: ```suggestion
        2. Benennen Sie den Herausgeber der Zeitschrift `<editor>`.
```

>         3. Ergänzen Sie im Impressum den Verlag inkl. seiner Adresse im Element `<imprint>`. Der **Verlag** `<publisher>` enthält hierbei den **Namen der Firma / Organisation** `<orgName>`, samt ihrer **Adresse** `<address>` bestehend aus einer oder mehrerer **Adresszeilen** `<addrLine>`.

Reviewer 2: ```suggestion
        3. Ergänzen Sie im Impressum `<imprint>` den Verlag `<publisher>` mit allen zur Verfügung stehenden Angaben:  neben dem **Namen der Organisation** `<orgName>` sollte auch die **Adresse** `<address>`, bestehend aus einer oder mehrerer **Adresszeilen** `<addrLine>`, enthalten sein.
```

>         4. Weisen Sie die Fa. 'L. Beck und Sohn' in einem 'responsibility statement' als Druckerei aus unter Verwendung von `<resp>`, `<orgName>`

Reviewer 2: ```suggestion
        4. Weisen Sie die Fa. 'L. Beck & Sohn' in einem 'responsibility statement' mit den Elementen `<resp>`, `<orgName>` als Druckerei aus.
```
=> Oder funktioniert das mit "&" nicht? Das wäre blöd, denn man sollte ja den Text entsprechend der Vorlage übernehmen. Wenn da was besonderes für "&" einzutippen ist, sollte man darauf in der Übungsangabe hinweisen.)

Author: Das "&" hat bis jetzt in jeder Form Probleme beim Rendering gemacht. Wir könnens noch mal testen aber ich würde fast sagen wir lassen es fürs erste beim "und" um Probleme mit der MDX Darstellung zu vermeiden.

Reviewer 2: Wie oben: Hier habe tatsächlich nur & verwendet - ich dachte im Fließtext ist das egal. Das müsste aber schon möglich sein, das wäre ja unangenehm wenn wir das nicht hinbekämen ...

Author: Guter Hinweis! 

Ja, im Fließtext ist das glaub ich nicht problematisch, hier gehts glaub ich eher darum, dass man das gleich schreibt wie in der Übung.

>         5. Geben Sie Wien als Publikationsort und das Erscheinungsjahr an `<pubPlace>`, `<date>`

Reviewer 2: ```suggestion
        5. Geben Sie Wien als Publikationsort und das Erscheinungsjahr mir den Elementen `<pubPlace>` und `<date>` an.
```

>         6. Kodieren Sie, dass der gewählte Textausschnitt aus der Nr. 31, 1924, Seite 11-12 stammt 'scope of bibliographic reference' `<biblScope>` + @type volume / page

Reviewer 2: ```suggestion
        6. Kodieren Sie, dass der gewählte Textausschnitt aus der "Nr. 31, 1924" stammt und spezifizieren Sie den Ausschnitt "Seite 11-12" im `<biblScope>`-Element mit dem `@unit`-Attribut und den Werten `volume` bzw. `page`.
```

      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>
</Quiz>

> 

Reviewer 2: ```suggestion
Der Vollständigkeit halber sei an dieser Stelle noch erwähnt, dass für die Metadatenerfassung von Handschriften und alten Drucken das Element `<msDesc>` **'manuscript description'** zur Verfügung steht.
```

Author: Soll hier noch zum TEI-C für weitere Dokumentation verlinkt werden?

Author: Ist die Zeile jetzt zwei mal drinnen?

Author: Dies scheint gelöst zu sein...

### Kodierung

> Der bei dieser Übung entstandene TEI-Header umfasst nur einige wesentliche Elemente, er kann im Bedarfsfall weitaus detaillierter ausfallen. Ergänzend kann dieser beispielhaft auch einige Angaben zur Annotation **'encoding description'** `<encodingDesc>` enthalten, die den Zusammenhang der physischen Ressource und des elektronischen Dokuments beschreibt. Diese reichen von Informationen zu Redaktionsprinzipien über Projektbeschreibungen, Angaben zur Verwendung von Tags bis hin zur  zugrundeliegenden Textauswahl.

Reviewer 2: ```suggestion
Der in diesen Übungen entstandene TEI-Header umfasst nur einige wesentliche Elemente, er kann im Bedarfsfall weitaus detaillierter ausfallen. Ergänzend können auch einige Angaben zur Annotation **'encoding description'** `<encodingDesc>` enthalten, die den Zusammenhang zwischen der physischen Ressource und dem elektronischen Dokument beschreibt wie z.B. Projektbeschreibungen, Redaktionsprinzipien, Angaben zur Verwendung von Tags oder der zu zugrundeliegenden Textauswahl und weitere mehr.
```

```xml
<encodingDesc>
  <editorialDecl>
    <p>Die Annotation folgt den Richtlinien der TEI/P5.</p>
    <hyphenation>
      <p>Zeilenenden sind mit dem Element 'lb' markiert. Trennungsstriche
        wurden entfernt und sind durch das Attribut @break mit dem Wert
        'no' repräsentiert.</p>
    </hyphenation>
    <normalization>
      <p>Typographie und Orthographie wurden weitestgehend beibehalten.</p>
    </normalization>
    <correction>
      <p>Vorgenommene Korrekturen wurden als solche annotiert.</p>
    </correction>
    <interpretation>
      <p>Personennamen historischer, religiöser, fiktionaler Figuren,
        Ortsnamen, Organisationsbezeichnungen, Datumsangaben</p>
    </interpretation>
  </editorialDecl>
</encodingDesc>
```

> Hier einige Beispiele für mögliche Angaben innerhalb der 'encoding description':

Reviewer 2: ```suggestion
Für unseren Beispieltext enthält die 'encoding description' folgende Unterelemente:
```

> - **editorische Angaben:** beschreibt die angewandten Richtlinien und Prinzipien für die Annotation `<editorialDecl>`.

Reviewer 2: ```suggestion
- `<editorialDecl>`
 - `<p>` enthält die allgemeine Angabe, dass nach den TEI Richtlinien in der Version P5 annotiert wurde.
 - `<hyphenation>`
 - `<normalization`
 - `<correction>`
 - `<interpretation>`
```

>   Wie in obenstehendem Beispiel können u.a. der Umgang mit Silbentrennung `<hyphenation>`, Normalisierung und Regulierung der originalen Quelle `<normalization>`, Korrekturen `<correction>` oder das Ausmaß an analytischer und interpretativer Information, die dem Text in der Transkription hinzugefügt wurde `<interpretation>`.

Reviewer 2: ```suggestion
```
=> Informationen wurden nicht in der "Transkription", sondern mit Markup hinzugefügt.

> - **Projektbeschreibung:** beschreibt das Ziel für das die Quelle kodiert wurde `<projectDesc>`

Reviewer 2: ```suggestion
```
=> Kommt im Beispiel gar nicht vor, deshalb gelöscht.

> - **Tagverwendung:** gibt informationen zum Tagging, das innerhalb des Dokuments angewendet wird `<tagsDecl>`

Reviewer 2: ```suggestion
```
=> Kommt im Beispiel gar nicht vor, deshalb gelöscht.

> - **Auswahlverfahren:** Fließtext, der das angewandte Samplingverfahren für einen Korpus oder eine Sammlung an Texten beschreibt `<samplingDecl>`

Reviewer 2: ```suggestion
```
=> Kommt im Beispiel gar nicht vor, deshalb gelöscht.

> #### Übung

Reviewer 2: ```suggestion
#### Quiz
```

<Quiz>
  <Quiz.Card>
    <Quiz.MultipleChoice>
      <Quiz.Question>

>         Welche Aspekte eines Dokuments werden in der Encoding Description eines Dokuments erfasst? (mehrere Antworten möglich)

Reviewer 2: ```suggestion
        Welche Aspekte eines Dokuments werden im TEI-Header  in der 'encoding description' erfasst? (mehrere Antworten möglich)
```

      </Quiz.Question>

      <Quiz.MultipleChoice.Option isCorrect>

>         editorische Angaben

Reviewer 2: ```suggestion
        Redaktionsprinzipien
```

      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>

>         Informationen über Glyphen und Beschreibung von nicht-standard Zeichenformaten, die im Quelltext verwendet werden

Reviewer 2: ```suggestion
        Informationen über den Umgang mit im Quelltext vorkommenden Sonderzeichen, insbesondere Nicht-Standard-Zeichen und Glyphen.
```

      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>

>         Projektbeschreibung

Reviewer 2: ```suggestion
        Projektbeschreibungen
```

      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>
        Tagverwendung
      </Quiz.MultipleChoice.Option>

      <Quiz.Message type="correct">

>         **Feedback:** Alle Antworten sind in diesem Falle richtig. Neben den im obenstehenden Text beschriebenen Angaben kann die Encoding-Description in der Character-Declaration `<charDecl>` auch Informationen zu nicht-standard Zeichenformaten und Glyphen enthalten.

Reviewer 2: ```suggestion
        **Feedback:** Alle Antworten sind in diesem Fall richtig. Neben den bereits genannten Angaben kann die 'encoding  description' u.a. auch **'character declarations'** `<charDecl>` enthalten um Informationen zum Umgang mit Nicht-Standard-Zeichen und Glyphen bereitzustellen.       
```
=> "**Feedback:**" wurde bisher nicht verwendet - vereinheitlichen? Zeichenformat passt nicht, denn in der analogen Quelle sind nur Zeichen, aber keine Formate enthalten.

      </Quiz.Message>
    </Quiz.MultipleChoice>
  </Quiz.Card>
</Quiz>

### Veröffentlichung

> Das **'publication statement'** `<publicationStmt>` gruppiert Informationen zur Publikation oder Distribution des elektronischen Texts, wie z.B. Informationen über den Herausgeber, den Erscheinungsort und das Erscheinungsdatum. Auch die Verfügbarkeit kann im TEI-Header enthalten sein, etwa unter welcher Lizenz das Dokument veröffentlicht wurde.

Reviewer 2: ```suggestion
Nachdem die Beschreibung der verwendeten Quelle abgeschlossen ist, gilt es, ebenso genaue Angaben zur Veröffentlichung des erstellten elektronischen Dokuments zu machen. Dazu kehren wir zurück zur bereits teilweise ausgefüllten 'file description' um im **'publication statement'** `<publicationStmt>` Informationen zur Publikation und Verbreitung des elektronischen Texts zu ergänzen. Neben dem Herausgeber, dem Erscheinungsort und -datum, können auch Angaben zur Verfügbarkeit gemacht werden.
```

Dies reicht vom unveröffentlichten Arbeitsdokument...

```xml
<publicationStmt>
  <p>Unveröffentlichtes Arbeitsdokument.</p>
</publicationStmt>
```

> ... bis hin zur unter einer Creative Commons Lizenz veröffentlichten autorisierten wissenschaftlichen Quelle.

Reviewer 2: ```suggestion
... oder ein Dokument mit der Lizenz "CC-BY 4.0" (Namensnennung 4.0 International) , das von anderen beliebig weiterverwendet werden darf, solange die Urheber\*innen/Rechteinhaber\*innen genannt werden. Eine andere Lizenz wäre "CC-BY-SA 4.0" (Namensnennung - Weitergabe unter gleichen Bedingungen 4.0 International). An diese Lizenz ist neben der Namensnennung als weitere Bedingung geknüpft, dass ein weiterverarbeitetes Dokument unter der gleichen Lizenz mit den gleichen Bedingungen zu versehen ist, sollte es publiziert werden.
```

Reviewer 2: => Bitte diesen Text übernehmen - Sidenote nicht notwendig.

```xml

> <publicationStmt>

Reviewer 2: ```suggestion
<publicationStmt>
```
=> => Was bedeutet das Zeichen hier? Taucht mehrmals auf.

  <publisher>

>     <orgName>Austrian Centre for Digital Humanities</orgName>

Reviewer 2: ```suggestion
    <orgName>Austrian Centre for Digital Humanities and Cultural Heritage</orgName>
```
=> Was bedeutet "\" hier mehrmals im Code?
=> Ich würde für das neue HowTo auch den aktuellen Institutsnamen verwenden - das alet Tutorial entstand tatsächlich noch im ACDH.

>     <email>acdh@oeaw.ac.at</email>

Reviewer 2: ```suggestion
    <email>acdh-ch@oeaw.ac.at</email>
```

    <idno type="VIAF">6515148451584915970000</idno>
  </publisher>

>   <pubPlace>Vienna</pubPlace>

Reviewer 2: ```suggestion
  <pubPlace>Wien</pubPlace>
```
=> Bisher alles auf Deutsch, daher auch das.

  <date>2017</date>
  <availability>
    <licence target="https://creativecommons.org/licenses/by-sa/4.0/">

>       <p>Veröffentlicht unter CC-BY-SA 4.0</p>

Reviewer 2: ```suggestion
      <p>Veröffentlicht unter CC-BY-SA 4.0</p>
```
=> Was bedeutet hier im Code ""?

>       <p>Veröffentlicht unter CC-BY-SA 4.0</p>

Reviewer 2: ```suggestion
      <p>Veröffentlicht unter CC-BY-SA 4.0</p>
```
=> Zeichen nach der schließenden Spitzklammer vorsorglich gelöscht.

>     </licence>

Reviewer 2: ```suggestion
    </licence>
```
=> Zeichen nach der schließenden Spitzklammer vorsorglich gelöscht.

  </availability>
</publicationStmt>
```

> Wie am ersten Beispiel ersichtlich reicht ein einfacher Absatz um eine Publikation zu beschreiben, oder aber man verwendet eine deutlich komplexere semantische Schreibweise. Einige Elemente wie `<publisher>`, `<orgName>`, `<pubPlace>` oder `<date>` im obigen Beispiel sind bereits bekannt.

Reviewer 2: ```suggestion
Wie ersichtlich würde ein einfacher Absatz ausreichen um eine Publikation zu beschreiben. Man kann aber auch hier eine deutlich komplexere semantische Strukturierung vornehmen. Einige Elemente wie `<publisher>`, `<orgName>`, `<pubPlace>` oder `<date>` sind bereits aus der Beschreibung der Quelle bekannt.
```

> Das Element **'electronic mail address'** enthält eine Emailadresse. Das Element **'identifier'** `<idno>` enthält eine Form der standardisierten Identifikation eines Objekts -- in unserem Fall eines bibliografischen Objekts. Das Attribut `type` gibt die Art der Identifikation an. Diese kann beispielsweise folgende sein:

Reviewer 2: ```suggestion
Als weitere Elemente stehen hier u.a. das `<email>`-Element **'electronic mail address'** für eine Emailadresse sowie das  Element **'identifier'** `<idno>`, das einen Bezeichner für die standardisierte Identifikation eines Objekts enthält. In diesem Fall ist es ein bibliografisches Objekt, konkret ein Bezeichner für das Institut, das als 'publisher' auftritt. Das Attribut `@type`, das die Art der Identifikation angibt, verweist auf den Eintrag für das ACDH-CH in den [Virtual Authority File(s)](https://viaf.org/viaf/6515148451584915970000/):
```

> 

Reviewer 2: ```suggestion
Andere Identifikatoren wären etwa:

```

> - **ISBN**: "International Standard Book Number" (ISBN-Nummer), die Bücher international eindeutig identifiziert.

Reviewer 2: ```suggestion
- **ISBN**: 'International Standard Book Number' (Internationale Standardbuchnummer) für die eindeutige Identifikation von Büchern.
```

> - **ISSN:** "International Standard Serial Number" (ISSN-Nummer), eine international einheitliche Indentifikationsnummer für serielle Publikationen, wie z.B. Zeitschriften.

Reviewer 2: ```suggestion
- **ISSN:** 'International Standard Serial Number' (Internationale Standardnummer für fortlaufende Sammelwerke) für die Identifikation von Zeitschriften und Schriftenreihen.
```

> - **DOI:** "Digital Object Identifier", eine einmalige Zeichenfolge (aus Nummern und Buchstaben) die ein elektronisches Dokument identifiziert.

Reviewer 2: ```suggestion
- **DOI:** "Digital Object Identifier", eine einmalige Zeichenfolge (aus Nummern und Buchstaben) für die Identifikation digitaler Objekte.
```

> - **URI:** Uniform Resource Identifier, eine eindeutige Identifikation einer Ressource (von physischen Objekte bis hin zu digitalen Ressourcen wie Webseiten), die gleichzeitig Hinweise auf den Zugriff der Ressource gibt.

Reviewer 2: ```suggestion
- **URI:** 'Uniform Resource Identifier' für Ressourcen im Internet. Das können einzelne Dokumente aber auch Webseiten, Webservices sein.
```

> - **VIAF:** Eine Normdatennummer im Virtual International Authority File (VIAF), einer virtuellen internationalen Normdatei als Verknüpfung nationaler Normdateien z.B. der gemeinsamen Normdatei. Diese wird als Uniform Resource Identifier für Linked-Open-Data Anwendungen genutzt.

Reviewer 2: ```suggestion
- **VIAF:** 'Virtual International Authority File', eine virtuelle internationale Normdatei die nationale Normdateien zusammenführt und identische Datensätze verlinkt. VIAF-Identifikatoren kommen nicht nur bei der Katalogisierung von Bibliotheks- und Archivbeständen zum Einsatz, sondern werden auch von Museen, Projekten und Web-Anwendungen genutzt.
```

> Das Element `<availability>` gibt Informationen zur Verfügbarkeit des Textes sowie Einschränkungen zur Nutzbarkeit, der Verbreitung, dem Copyright Status, oder der Lizenzierung. Die **Lizenzinformation** wird innerhalb des **availability-Elements** mittels des Elements `<licence>` angegeben. Das Attribut `target` wird hierbei zur Referenzierung des Volltextes der Lizenz verwendet.

Reviewer 2: ```suggestion
Schließlich enthält das Element `<availability>` Informationen zur Verfügbarkeit eines Textes, benennt Nutzungs- und Distributionseinschränkungen, gibt den Status des Urheberrechts und darüber hinaus die Lizenzierung an. Das Attribut `@target` kann neben anderen im `<licence>`-Element verwendet werden um auf den Volltext der Lizenz zu verweisen.
```

> #### Übung

Reviewer 2: ```suggestion
#### Quiz
```

<Quiz>
  <Quiz.Card>
    <Quiz.MultipleChoice>
      <Quiz.Question>

>         Was wird im Publication Statement dokumentiert?

Reviewer 2: ```suggestion
        Was wird im 'publication statement' dokumentiert?
```

      </Quiz.Question>

      <Quiz.MultipleChoice.Option>

>         Das Publication Statement enhält die Angaben zur Publikation des originalen Textes in Print.

Reviewer 2: ```suggestion
        Das 'publication statement' enthält Publikationsangaben zur gedruckten Textvorlage.
```

      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>

>         Das Publication Statement enhält alle Informationen rund um die Publikation und Distribution des elektronischen Texts.

Reviewer 2: ```suggestion
        Das 'publication statement' enthält alle Informationen rund um die Publikation und Distribution des elektronischen Texts.
```

      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option>

>         Das Publication Statement enhält das Vorwort des Herausgebers zur Publikation.

Reviewer 2: ```suggestion
        Das 'publication statement' enthält das Vorwort des Herausgebers zur Publikation.
```

      </Quiz.MultipleChoice.Option>
    </Quiz.MultipleChoice>
  </Quiz.Card>
</Quiz>

### Provenienz

> Die **'revision description'** `<revisionDesc>` fasst die Revisionsgeschichte einer Datei zusammen und wird zumeist ans Ende des Headers gestellt. "Wer hat wann was geändert?" wird in `<change>` Statements wie folgt festgehalten:

Reviewer 2: ```suggestion
Die **'revision description'** `<revisionDesc>` fasst die Revisionsgeschichte einer TEI-Datei zusammen. "Wer wann welche Änderungen vorgenommen hat", wird in den Elementen `<change>` bzw. `<listChange>` festgehalten:
```
=> "zumeist" stimmt hier nicht, die Elemente im TEI-Header müssen in einer bestimmten Reihenfolge angeordnet sein.
**=> ... in progress ...**

Reviewer: Um der Darstellung im Element zu entsprechen, würde ich vorschlagen "Wann wer welche Änderungen vorgenommen hat"...

Reviewer 2: => Gute Idee - ist geändert - bitte übernehmen

```xml
<revisionDesc>

>   <change when="2017-04-04" who="#uc #ds" status="published">Freigegeben.</change>

Reviewer 2: ```suggestion
  <change when="2017-04-04" who="#em #mm" status="published">Freigegeben.</change>
```

>   <change when="2017-04-03" who="#uc" status="revised">Korrektur.</change>

Reviewer 2: ```suggestion
  <change when="2017-04-03" who="#em" status="revised">Korrektur.</change>
```

>   <change when="2017-04-01" who="#uc" status="draft">Einleitung überarbeitet.</change>

Reviewer 2: ```suggestion
  <change when="2017-04-01" who="#em" status="draft">Einleitung überarbeitet.</change>
```

>   <change when="2017-03-28" who="#ds" status="draft">Erste Version fertig.</change>

Reviewer 2: ```suggestion
  <change when="2017-03-28" who="#mm" status="draft">Erste Version fertig.</change>
```

</revisionDesc>
```

> Das Attribut `when` im change-Statement enhält das Datum der Änderung, der `status` den Status des Dokuments (also Entwurf, Revision, publiziert), `who` verweist auf eine Person, die an der Änderung beteiligt oder für diese verantwortlich ist, und die für gewöhnlich im 'responsibility statement' oder in einem `<person>`-Element an anderer Stelle im Dokument näher bezeichnet wird.

Reviewer 2: ```suggestion
Die im `<change>`-Element angegebenen Attribute und ihre Werte dienen der Spezifizierung:  `@when` enthält das Datum der Änderung, `@who` nennt die für die Änderung verantwortliche Person, wobei die Raute vor dem Bezeichner auf die Definition dieser Person verweist, die sich gewöhnlich im 'statement of responsibility' des TEI-Header befindet:

```xml
<!-- ... -->
<persName xml:id="em">Erika Musterfrau</persName>
<persName xml:id="em">Max Mustermann</persName>
```

Die Werte Attribut `@status` geben den Status des Dokuments nach der Änderung an, z.B. ob es sich um einen Entwurf, eine fertige Korrektur oder bereits für die Veröffentlichung freigegebenen Version handelt. Die Reihenfolge der Änderungen sollte chronologisch entweder auf- oder absteigend sortiert sein.
```

> 

Reviewer 2: ```suggestion

```

> Die neueste Veränderung am Dokument sollte hierbei zuerst aufgeführt werden.

Reviewer 2: ```suggestion
```suggestion
<!-- ... -->
<persName xml:id="em">Erika Musterfrau</persName>
<persName xml:id="em">Max Mustermann</persName>
<!-- ... -->
```
=> Bitte als ```xml-Beispiel einfüfen ... bekomme ich hier nicht hin
```

#### Übung

<Quiz>
  <Quiz.Card>
    <Quiz.XmlCodeEditor

>       code={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt - ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n        <respStmt>\n          <resp>TEI Kodierung</resp>\n          <persName>Maxi Musterfrau</persName>\n        </respStmt>\n      </titleStmt>\n      <publicationStmt>\n        <p>Publication Information</p>\n      </publicationStmt>\n      <sourceDesc>\n        <biblStruct>\n          <monogr>\n            <title type=\"main\">Die Filmwelt</title>\n            <title type=\"sub\">Illustrierte Kino Revue</title>\n            <editor>Felix Brasch</editor>\n            <imprint>\n              <publisher>\n                <orgName>Zeitungsverlag Universale Ges. m. b. H.</orgName>\n                <address>\n                  <addrLine>Wien, VII., Hermanngasse 5</addrLine>\n                </address>\n              </publisher>\n              <respStmt>\n                <resp>Druckerei</resp>\n                <orgName>L. Beck und Sohn</orgName>\n              </respStmt>\n              <pubPlace>Wien</pubPlace>\n              <date>1924</date>\n            </imprint>\n            <biblScope unit=\"volume\">31, 1924</biblScope>\n            <biblScope unit=\"page\">11-12</biblScope>\n          </monogr>\n        </biblStruct>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Some text here.</p>\n    </body>\n  </text>\n</TEI>"}

Reviewer 2: ```suggestion
      code={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt – ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n        <respStmt>\n          <resp>TEI-Kodierung</resp>\n          <persName>Erika Musterfrau</persName>\n        </respStmt>\n      </titleStmt>\n      <publicationStmt>\n        <p>Informationen zur Veröffentlichung</p>\n      </publicationStmt>\n      <sourceDesc>\n        <biblStruct>\n          <monogr>\n            <title type=\"main\">Die Filmwelt</title>\n            <title type=\"sub\">Illustrierte Kino Revue</title>\n            <editor>Felix Brasch</editor>\n            <imprint>\n              <publisher>\n                <orgName>Zeitungsverlag Universale Ges. m. b. H.</orgName>\n                <address>\n                  <addrLine>Wien, VII., Hermanngasse 5</addrLine>\n                </address>\n              </publisher>\n              <respStmt>\n                <resp>Druckerei</resp>\n                <orgName>L. Beck &amp; Sohn</orgName>\n              </respStmt>\n              <pubPlace>Wien</pubPlace>\n              <date>1924</date>\n            </imprint>\n            <biblScope unit=\"volume\">31, 1924</biblScope>\n            <biblScope unit=\"page\">11-12</biblScope>\n          </monogr>\n        </biblStruct>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text.</p>\n    </body>\n  </text>\n</TEI>"}
```

Reviewer 2: => Nach Klärung der "&"-Frage, bitte übernehmen.

>       solution={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt - ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n        <respStmt>\n          <resp>TEI Kodierung</resp>\n          <persName>Maxi Musterfrau</persName>\n        </respStmt>\n      </titleStmt>\n      <publicationStmt>\n        <p>Publication Information</p>\n      </publicationStmt>\n      <sourceDesc>\n        <biblStruct>\n          <monogr>\n            <title type=\"main\">Die Filmwelt</title>\n            <title type=\"sub\">Illustrierte Kino Revue</title>\n            <editor>Felix Brasch</editor>\n            <imprint>\n              <publisher>\n                <orgName>Zeitungsverlag Universale Ges. m. b. H.</orgName>\n                <address>\n                  <addrLine>Wien, VII., Hermanngasse 5</addrLine>\n                </address>\n              </publisher>\n              <respStmt>\n                <resp>Druckerei</resp>\n                <orgName>L. Beck und Sohn</orgName>\n              </respStmt>\n              <pubPlace>Wien</pubPlace>\n              <date>1924</date>\n            </imprint>\n            <biblScope unit=\"volume\">31, 1924</biblScope>\n            <biblScope unit=\"page\">11-12</biblScope>\n          </monogr>\n        </biblStruct>\n      </sourceDesc>\n    </fileDesc>\n    <revisionDesc>\u000b\n      <change when=\"2021-04-21\" who=\"#mm\" status=\"draft\">Erster Entwurf abgeschlossen.</change>\n    </revisionDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Some text here.</p>\n    </body>\n  </text>\n</TEI>"}

Reviewer 2: ```suggestion
      solution={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt – ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n        <respStmt>\n          <resp>TEI-Kodierung</resp>\n          <persName>Erika Musterfrau</persName>\n        </respStmt>\n      </titleStmt>\n      <publicationStmt>\n        <p>Informationen zur Publikation</p>\n      </publicationStmt>\n      <sourceDesc>\n        <biblStruct>\n          <monogr>\n            <title type=\"main\">Die Filmwelt</title>\n            <title type=\"sub\">Illustrierte Kino Revue</title>\n            <editor>Felix Brasch</editor>\n            <imprint>\n              <publisher>\n                <orgName>Zeitungsverlag Universale Ges. m. b. H.</orgName>\n                <address>\n                  <addrLine>Wien, VII., Hermanngasse 5</addrLine>\n                </address>\n              </publisher>\n              <respStmt>\n                <resp>Druckerei</resp>\n                <orgName>L. Beck &amp; Sohn</orgName>\n              </respStmt>\n              <pubPlace>Wien</pubPlace>\n              <date>1924</date>\n            </imprint>\n            <biblScope unit=\"volume\">31, 1924</biblScope>\n            <biblScope unit=\"page\">11-12</biblScope>\n          </monogr>\n        </biblStruct>\n      </sourceDesc>\n    </fileDesc>\n    <revisionDesc>\u000b\n      <change when=\"2021-04-21\" who=\"#mm\" status=\"draft\">Erster Entwurf abgeschlossen.</change>\n    </revisionDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text.</p>\n    </body>\n  </text>\n</TEI>"}
```

Reviewer 2: => Nach Klärung der "&"-Frage, bitte übernehmen.

      validate="input"
    >
      <Quiz.Question>

>         Fügen Sie dem bisherigen Dokument den ersten Eintrag der Revision History mit dem Datum 21.04.2021 und dem Namenskürzel von Maxi Musterfrau (mm) als Autorin hinzu.

Reviewer 2: ```suggestion
        Fügen Sie hier einen ersten Eintrag der Revisionsgeschichte mit dem Datum 21.04.2021 und dem Namenskürzel von Max Mustermann "mm" als für Urheber der Änderung hinzu.
```

>         Markieren Sie das Dokument als Entwurf `<draft>` und beschreiben Sie den Status als "Erster Entwurf abgeschlossen".

Reviewer 2: ```suggestion
        Setzen Sie im `@status`-Attribut den Wert **`draft`** und als Beschreibungstext als **"Erster Entwurf abgeschlossen."** ein
```

      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>
</Quiz>

### Inhalt & Kontext

> Auch Informationen zum Inhalt des elektronischen Dokuments und seinem Kontext können bereits im Header erfasst werden. Hierfür kommt die **'text-profile description'** `<profileDesc>` zum Einsatz, die nicht-bibliographische Aspekte des Textes wie Sprachen, Produktionskontext oder die vorkommenden Personen / Stimmen erfasst.

Reviewer 2: ```suggestion
Eine der eingangs erwähnten fünf Hauptkomponenten des TEI-Header bedarf noch näherer Erläuterungen: Die **'text-profile description'** `<profileDesc>` kommt zum Einsatz um nicht-bibliografische Aspekte eines Textes wie die verwendeten Sprachen, den Produktionskontext oder die daran Beteiligten und ihr Umfeld zu beschreiben.
```

> Innerhalb der 'text-profile description' können beispielsweise folgende Aspekte mit dem jeweils nachstehenden Element erfasst werden:

Reviewer 2: ```suggestion
```
=> In Liste integriert.

> Innerhalb der 'text-profile description' können beispielsweise folgende Aspekte mit dem jeweils nachstehenden Element erfasst werden:

Reviewer 2: ```suggestion
Im Element `<profileDesc>` stehen u.a. diese Unterelemente zur Verfügung:
```

> - Sprachen im Text `<langUsage>`

Reviewer 2: ```suggestion
- `<abstract>`: enthält eine Zusammenfassung
- `<correspDesc>` ('correspondence description'): erfasst Korrespondenzbeschreibungen z.B. bei Brief-Editionen
- `<creation>`: erläutert die Textproduktion
- `<langUsage>` ('language usage'): beschreibt Sprachen, Varietäten, Dialekte usw.
- `<textClass>` ('text classification'): ermöglicht die Verschlagwortung

```
=> Punkte darunter gelöscht. Hier umsortiert und an die anderen Listen in diesem abschnitt angepasst.

> - Verschlagwortung `<textClass>`

Reviewer 2: ```suggestion
```

> - Entstehung `<creation>`

Reviewer 2: ```suggestion
```

> - Zusammenfassung `<abstract>`

Reviewer 2: ```suggestion
```

> - Briefe `<correspDesc>`

Reviewer 2: ```suggestion
```

> - Personen im Text `<particDesc>`

Reviewer 2: ```suggestion
```

#### Übung

<Quiz>
  <Quiz.Card>
    <Quiz.MultipleChoice>
      <Quiz.Question>

>         Welche Aspekte eines Textes werden in der 'Text Profile Description' erfasst? (mehrere Antworten möglich)

Reviewer 2: ```suggestion
        Welche Aspekte eines Textes werden in der 'text-profile description' erfasst? (mehrere Antworten möglich)
```

      </Quiz.Question>

      <Quiz.MultipleChoice.Option>

>         Bibliographische Angaben zu einem Text.

Reviewer 2: ```suggestion
        Bibliografische Angaben zu einem Text.
```

      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>

>         Nicht-bibliographische Aspekte des Textes.

Reviewer 2: ```suggestion
        Nicht-bibliografische Aspekte eines Textes.
```

      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>

>         Informationen zum Inhalt des elektronischen Dokuments, z.B. im Text vorkommende Personen, Sprachen, o.ä.

Reviewer 2: ```suggestion
        Inhaltliche Aspekte wie z.B. im Text vorkommende Personen, Sprachen, u.ä.
```

      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>

>         Verschlagwortung der Textsorte oder des Themas eines Textes mittels eines standardisierten Klassifikationsschemas oder Thesaurus.

Reviewer 2: ```suggestion
        Thematische Aspekte, im Sinne einer Verschlagwortung mit Hilfe eines standardisierten Klassifikationsschemas oder Thesaurus.
```

      </Quiz.MultipleChoice.Option>

      <Quiz.Message type="correct">

>         **Feedback:** Die 'Text Profile Description' enthält keine bibliographischen Angaben zum Text. Nicht-bibliographische Aspekte, wie auch Informationen zum Inhalt oder Verschlagwortungen zum Text können aber enthalten sein.

Reviewer 2: ```suggestion
        **Feedback:** Die 'Text Profile Description' enthält keine bibliographischen Angaben zum Text. Nicht-bibliographische Aspekte, wie auch Informationen zum Inhalt oder Verschlagwortungen zum Text können aber enthalten sein.
```
=> Wie Zeile 278: "Feedback:" wurde bisher nicht verwendet – vereinheitlichen? Entweder hier löschen bzw. überall sonst hinzufügen. Ich wäre für löschen.
=> Bei "Falsch" kommt nur "Falsch", aber keine Erklärung keine Erklärung gegeben.
=> Bei korrekter Beantwortung fehlt "Richtig!" ... ist das Absicht?
=> + Soll "Falsch/Richtig" fett sein und in eigener Zeile stehen? Rufzeichen oder Punkt? ... checken ...

      </Quiz.Message>
    </Quiz.MultipleChoice>
  </Quiz.Card>
</Quiz>

## Weiterführende Information

> Eine tiefergehende Einführung zum TEI Header gibt es [auf der Seite der Text Encoding Initiative](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/HD.html#HD21).

Reviewer 2: ```suggestion
Das entsprechende Modul zum TEI-Header auf den Webseiten der Text Encoding Initiative finden sie [hier](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/HD.html).
```
=> Das ist keine Einführung, sondern die "Bibel" ;-) - hier sollte auch nicht zu einem Unterpunkt, sondern zum ganzen Kapitel verlinkt werden.

Für eine vollständige Strukturierung aller Teile wie sie im TEI-Header wünschenswert ist, empfiehlt sich die Verwendung von **'structured bibliographic citation'** `<biblStruct>`, dessen Unterelemente allerdings nicht der Satzstruktur einer Vorlage folgen, sondern in einer bestimmten Reihenfolge angeordnet sind:

```xml
<biblStruct>
  <monogr>
    <editor>Matthew K. Gold</editor>
    <title>Debates in the Digital Humanities</title>
    <imprint>
      <publisher>University of Minnesota Press</publisher>
      <pubPlace>Minneapolis and London</pubPlace>
      <date>2012</date>
    </imprint>
  </monogr>
</biblStruct>
```

Hierbei wird innerhalb der **'structured bibliographic citation'** ein Buch oder eine Zeitschrift mit dem Element **'monographic level'** `<monogr>` umschlossen und die Information so gruppiert. Innerhalb dessen können Autorinnen und Autoren _mit dem `<author>` sowie Editorinnen und Editoren_ eines Buches oder einer Zeitschrift mit dem Element `<editor>` benannt werden.

> Der **'scope of bibliographic reference'** `<biblScope>`definiert, auf welchen Teil einer Arbeit sich die Referenz bezieht. Dies können z.B. bestimmte Seiten sein, die mit dem Attribut `unit="page"` angegeben werden, oder ein bestimmter Jahrgang einer Zeitschrift `unit="volume"`.

Author: müsste hier im Sinne der Konvention auch wieder ein @unit stehen?

Reviewer 2: Ja stimmt - völlig falsch auch die Anführungszeichen ... bitte ausbessern zu: `@unit=page` und `unit=volume`

        **Zu verwendende Elemente / Attribute / Werte:**

        1. Geben Sie Haupt- und Untertitel der Zeitschrift an `<title>` + `@type=main/sub`.
        2. Benennen Sie den Herausgeber der Zeitschrift `<editor>`.
        3. Ergänzen Sie im Impressum `<imprint>` den Verlag `<publisher>` mit allen zur Verfügung stehenden Angaben: neben dem Namen der Organisation `<orgName>` sollte auch die Adresse `<address>`, bestehend aus einer Adresszeile `<addrLine>`, enthalten sein.
        4. Weisen Sie die Fa. 'L. Beck und Sohn' in einem 'responsibility statement' als Druckerei aus unter Verwendung von `<resp>`, `<orgName>`
        5. Geben Sie "Wien" als Publikationsort und "1924" als Erscheinungsjahr mit den Elementen `<pubPlace>` und `<date>` an.
        6. Kodieren Sie, dass der gewählte Textausschnitt aus der Nr. "31, 1924" stammt und spezifizieren Sie den Ausschnitt Seite "11-12" in zwei `<biblScope>`-Elementen mit dem `@unit`-Attribut und dem Wert `volume` bzw. `page`.
      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>
</Quiz>

> Der Vollständigkeit halber sei an dieser Stelle noch erwähnt, dass für die bibliografische Strukturierung von Handschriften und frühen Drucken das Element **'manuscript description'** `<msDesc>` für die ausführliche Beschreibung im TEI-Header zur Verfügung steht.

Author: ```suggestion
Der Vollständigkeit halber sei an dieser Stelle noch erwähnt, dass für die bibliografische Strukturierung von Handschriften und frühen Drucken das Element **'manuscript description'** `<msDesc>` für die ausführliche Beschreibung im TEI-Header zur Verfügung steht.

```

=> soll hier noch auf TEI-C für weitere Dokumentation zu msDesc verlinkt  werden?

Reviewer 2: Ja gerne, gute Idee!

- `<p>` enthält die allgemeine Information, dass nach den TEI-Richtlinien in der Version P5 annotiert wurde.
- `<hyphenation>`: beschreibt den Umgang mit Worttrennungen am Zeilenende.
- `<normalization>`: beschreibt, inwieweit Typo- und Orthografie normalisiert wurden.
- `<correction>`: gibt an, ob Korrekturen stillschweigend vorgenommen, oder als solche annotiert wurden.
- `<interpretation>`: listet auf, welche semantischen Aspekte annotiert wurden, d.h. welche analytischen oder interpretierenden Informationen dem Text **zusätzlich** zur Transkription hinzugefügt wurden.
 
Die oben erwähnten, jedoch im Beispiel aus Gründen der Übersichtlichkeit nicht enthaltenen weiteren Unterelemente, sind:
 - `<projectDesc>` ('project description'): beschreibt das Projekt, in dessen Rahmen die Quelle(n) digitalisiert und annotiert wurde(n).
 - `<tagsDecl>` ('tagging declaration'): beschreibt die angewendete Tagging-Technik.
 - `<samplingDecl>` ('sampling declaration'): beschreibt die Gründe für die Textauswahl und die dabei angewandten Methoden. 

> - **Auswahlverfahren:** Fließtext, der das angewandte Samplingverfahren für einen Korpus oder eine Sammlung an Texten beschreibt `<samplingDecl>`

Reviewer: ```suggestion

```

    >
      <Quiz.Question>
        Fügen Sie hier einen ersten Eintrag der Revisionsgeschichte mit dem Datum 21.04.2021 und dem Namenskürzel von Max Mustermann "mm" als den Urheber dieser Änderung hinzu.

        Setzen Sie im `@status`-Attribut den Wert **`draft`** und als Beschreibungstext **"Erster Entwurf abgeschlossen."** ein
      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>
</Quiz>

### Inhalt & Kontext

Eine letzte der eingangs erwähnten fünf Hauptkomponenten des TEI-Header, die hier näher erläutert werden soll, ist die **'text-profile description'** `<profileDesc>`, die zum Einsatz kommt um nicht-bibliografische Aspekte von Texten wie z.B. die verwendeten Sprachen, den Produktionskontext, die daran Beteiligten und ihren Hintergrund zu beschreiben, oder im Text vorkommende Personen zu erfassen.

Im Element `<profileDesc>` stehen u.a. diese Unterelemente zur Verfügung:

- `<abstract>`: enthält eine Zusammenfassung.
- `<correspDesc>` ('correspondence description'): erfasst Korrespondenzbeschreibungen für Brief-Editionen.
- `<creation>`: erläutert die Textproduktion.
- `<langUsage>` ('language usage'): beschreibt Sprachen, Varietäten, Dialekte usw.

> - `<particDesc>` ('participation description'): beschreibt sowohl im Text als auch den Metadaten vorkommende Sprecher, "Stimmen" und Teilnehmer.

Reviewer: Hier müsste man noch gendern.

Reviewer 2: => Danke für den Hinweis - geändert - bitte übernehmen.

---
title: 4. TEI-Header
lang: de
date: 2021-07-14T14:38:49.988Z
version: 1.0.0
authors:
  - czeitschner-ulrike
  - schopper-daniel
editors:
  - königshofer-elisabeth
  - wiencek-florian
tags:
  - tei
  - xml
abstract: Dieses Kapitel stellt den TEI-Header und seine Komponenten vor, die
  dazu dienen, ein kodiertes TEI-Dokument detailliert zu beschreiben.
licence: ccby-4.0
toc: false
uuid: j51QIatS7jhSnysibO4Cs
---
## Lerninhalte

- Metadaten für TEI-Dokumente
- Komponenten und Strukturierung des TEI-Header

## Grundlagen

Im `<teiHeader>`-Element, das an erster Stelle innerhalb des Wurzelelements `<TEI>` positioniert ist, werden die Metadaten _über_ ein TEI-Dokument erfasst. Für die detaillierte Beschreibung der für den Datenaustausch und die -publikation relevanten Informationen steht eine Vielzahl an Elementen zur Verfügung. 

Der TEI-Header ist in jedem TEI-Dokument **obligatorisch** zu verwenden. Er kann nur einige wesentliche Metadaten enthalten, bei Bedarf jedoch sehr detailreich sein. Seine fünf Hauptkomponenten sind:

- `<fileDesc>` ('file description'): beschreibt das elektronische Dokument.
- `<encodingDesc>` ('encoding description'): beschreibt Kodierungsrichtlinien.

> - `<profileDesc>` ('text-profile description'): gibt Kontextinformationen zum Text wie die Entstehungszeit, Sprache oder Textsorte.

Reviewer 2: ```suggestion
- `<profileDesc>` ('text-profile description'): gibt Kontextinformationen zum Text wie die Entstehungszeit, Sprache oder Textsorte an.
```

<?xml-model href="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/tei_all.rng" type="application/xml"
	schematypens="http://purl.oclc.org/dsdl/schematron"?>
<TEI xmlns="http://www.tei-c.org/ns/1.0">
  <teiHeader>
    <fileDesc>
      <titleStmt>
        <title>Titel</title>
      </titleStmt>
      <publicationStmt>
        <p>Informationen zur Veröffentlichung</p>
      </publicationStmt>
      <sourceDesc>
        <p>Informationen über die Quelle</p>
      </sourceDesc>
    </fileDesc>
  </teiHeader>
  <text>
    <body>
      <p>Text.</p>
    </body>
  </text>
</TEI>
```

Da die Metadaten im TEI-Header ein elektronisches Dokument so ausführlich wie möglich beschreiben sollen, empfiehlt es sich, neben den hier angeführten Pflichtangaben auch einige der zur Verfügung stehenden optionalen Elemente zu verwenden. Als Vorlage für die konkreten Übungen dieses Abschnitts dient der in Kapitel 2 **\[Link zu Kapitel 2]** vorgestellte Beispieltext _Die Filmwelt_. Die für die Befüllung der Elemente notwendigen Informationen sind bei der jeweiligen Übung angegeben.

> Im Folgenden gehen wir auf die wichtigsten Metadaten-Informationen ein, die auch für unser Beispiel der _Filmwelt_ relevant sind. In einer Übung werden dann die Informationen der zwei Seiten aus der _Filmwelt_ entsprechend kodiert.

Reviewer 2: ```suggestion
```
=> Gelöscht, weil redundant. Dass die Filmwelt kodiert wird und es dazu Übungen gibt, geht bereits aus dem vorhergehendem Absatz hervor.

      solution={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt – ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n        <respStmt>\n          <resp>TEI-Kodierung</resp>\n          <persName>Erika Musterfrau</persName>\n        </respStmt>\n      </titleStmt>\n      <publicationStmt>\n        <p>Informationen zur Veröffentlichung</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Informationen über die Quelle</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text.</p>\n    </body>\n  </text>\n</TEI>"}
      validate="input"
    >
      <Quiz.Question>

>         2. Ergänzen Sie bitte im **'title statement'** das **'statement of responsibility'** um folgende Elemente und Informationen:

Reviewer 2: => Stimmt da bei der 2. Frage was nicht? Egal was ich eingebe, es kommt immer "Falsch" ... kann nicht herausfinden, ob es an mir liegt ... sorry.

Author: Es war in der Code-Angabe Publikation und in der Lösung Veröffentlichung im Publication Statement gestanden. Wenn man etwas ausbessert muss man es immer für beide ausbessern. Ich habe es mal auf Publikation bei beiden abgeändert. Ich hoffe das passt so. Sonst bitte noch mal bei beiden ändern.

Reviewer 2: => Habe das leider übersehen ... tut mir leid. Habe es 2x auf "Veröffentlichung" geändert, weil es überall sonst auch so ist.

```xml
<bibl><editor>Matthew K. Gold</editor>, ed., <date>2012</date>. <title>Debates in the Digital Humanities.</title> <pubPlace>Minneapolis</pubPlace>, <pubPlace>London</pubPlace>: <publisher>University of Minnesota Press</publisher>.</bibl>
```

Diese Auszeichnungsvariante ist nicht auf den TEI-Header beschränkt, sondern kann auch außerhalb verwendet werden. Sie eignet sich z.B. besonders für die Annotation von Bibliografien mit vielen Literaturzitaten, deren "Satzstruktur" der Textvorlage entsprechend abgebildet werden soll.

Für eine vollständige Strukturierung aller Teile wie sie im TEI-Header wünschenswert ist, empfiehlt sich die Verwendung von **'structured bibliographic citation'** `<biblStruct>`, dessen Unterelemente allerdings nicht der Satzstruktur einer Vorlage folgen, sondern in einer bestimmten Reihenfolge angeordnet sind:

```xml
<biblStruct>
  <monogr>
    <editor>Matthew K. Gold</editor>
    <title>Debates in the Digital Humanities</title>
    <imprint>
      <publisher>University of Minnesota Press</publisher>
      <pubPlace>Minneapolis and London</pubPlace>
      <date>2012</date>
    </imprint>
  </monogr>
</biblStruct>
```

> Hierbei wird innerhalb der **'structured bibliographic citation'** ein Buch oder eine Zeitschrift mit dem Element **'monographic level'** `<monogr>` umschlossen und die Information so gruppiert. Innerhalb dessen können Autorinnen und Autoren _mit dem `<author>` sowie Editorinnen und Editoren_ eines Buches oder einer Zeitschrift mit dem Element `<editor>` benannt werden.

Reviewer 2: ```suggestion
Im `<biblStruct>`-Element gruppiert das Element **'monographic level'** `<monogr>` alle bibliografischen Angaben zu einem einzelnen Werk, wie ein ein Buch oder die Ausgabe einer Zeitschrift. Autor\*innen und Herausgeber\*innen werden mit den Elementen `<author>` bzw. `<editor>` annotiert. Dabei könnten zusätzlich die Elemente `<persName>`, `<forename>`, `<surname>` etc. zum Einsatz kommen. Letztere wurden hier aus Gründen der Übersichtlichkeit nicht verwendet. Es folgt der Titel und daran anschließend das `<imprint>`-Element, das Informationen zur Veröffentlichung und Verbreitung wie z.B. Verlag, Publikationsort und -datum zusammenfasst.
```
=> Bitte auch diesen Text übernehmen - habe das auch bei den gelösten Konversationen kommentiert.

        **Zu verwendende Elemente / Attribute / Werte:**

        1. Geben Sie Haupt- und Untertitel der Zeitschrift an `<title>` + `@type=main/sub`.
        2. Benennen Sie den Herausgeber der Zeitschrift `<editor>`.
        3. Ergänzen Sie im Impressum `<imprint>` den Verlag `<publisher>` mit allen zur Verfügung stehenden Angaben: neben dem Namen der Organisation `<orgName>` sollte auch die Adresse `<address>`, bestehend aus einer Adresszeile `<addrLine>`, enthalten sein.
        4. Weisen Sie die Fa. 'L. Beck und Sohn' in einem 'responsibility statement' als Druckerei aus unter Verwendung von `<resp>`, `<orgName>`
        5. Geben Sie "Wien" als Publikationsort und "1924" als Erscheinungsjahr mit den Elementen `<pubPlace>` und `<date>` an.
        6. Kodieren Sie, dass der gewählte Textausschnitt aus der Nr. "31, 1924" stammt und spezifizieren Sie den Ausschnitt Seite "11-12" in zwei `<biblScope>`-Elementen mit dem `@unit`-Attribut und dem Wert `volume` bzw. `page`.
      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>
</Quiz>

> Der Vollständigkeit halber sei an dieser Stelle noch erwähnt, dass für die bibliografische Strukturierung von Handschriften und frühen Drucken das Element **'manuscript description'** `<msDesc>` für die ausführliche Beschreibung im TEI-Header zur Verfügung steht (mehr Informationen hierzu in der [TEI-Dokumentation](https://tei-c.org/release/doc/tei-p5-doc/en/html/MS.html#msov).

Reviewer 2: ```suggestion
Der Vollständigkeit halber sei an dieser Stelle noch erwähnt, dass für die bibliografische Strukturierung von Handschriften und frühen Drucken das Element **'manuscript description'** `<msDesc>` für die ausführliche Beschreibung im TEI-Header zur Verfügung steht (\> Mehr Informationen hierzu in der [TEI-Dokumentation](https://tei-c.org/release/doc/tei-p5-doc/en/html/MS.html#msov)).
```
=> An Verweise bisher angepasst.

### Kodierung

> Der in diesen Übungen entstandene TEI-Header umfasst nur einige wesentliche Elemente, er kann im Bedarfsfall weitaus detaillierter ausfallen. Ergänzend können auch Informationen zur Kodierung in einer **'encoding description'** `<encodingDesc>` enthalten sein, die Auskunft über den Zusammenhang zwischen der Quelle und dem daraus abgeleiteten elektronischen Text geben wie z.B. Redaktionsprinzipien, Projektbeschreibungen, Angaben zur Verwendung von Tags, der zugrundeliegenden Textauswahl usw.:

Reviewer 2: ```suggestion
Der in diesen Übungen entstandene TEI-Header umfasst nur einige wesentliche Elemente, er kann im Bedarfsfall weitaus detaillierter ausfallen. Ergänzend können auch Informationen zur Kodierung in einer **'encoding description'** `<encodingDesc>` enthalten sein, die Auskunft über den Zusammenhang zwischen der Quelle und dem daraus abgeleiteten elektronischen Text geben wie z.B. Redaktionsprinzipien, Projektbeschreibungen, Angaben zur Verwendung von Tags, der Behandlung von Sonderzeichen, der zugrundeliegenden Textauswahl usw.:
```

    </interpretation>
  </editorialDecl>
</encodingDesc>
```

Für unseren Beispieltext enthält das `<encodingDesc>`-Element eine beispielhafte **'editorial declaration'** `<editorialDecl>` mit folgenden Unterelementen:

- `<p>` enthält die allgemeine Information, dass nach den TEI-Richtlinien in der Version P5 annotiert wurde.
- `<hyphenation>`: beschreibt den Umgang mit Worttrennungen am Zeilenende.
- `<normalization>`: beschreibt, inwieweit Typo- und Orthografie normalisiert wurden.
- `<correction>`: gibt an, ob Korrekturen stillschweigend vorgenommen, oder als solche annotiert wurden.
- `<interpretation>`: listet auf, welche semantischen Aspekte annotiert wurden, d.h. welche analytischen oder interpretierenden Informationen dem Text **zusätzlich** zur Transkription hinzugefügt wurden.

Die oben erwähnten, jedoch im Beispiel aus Gründen der Übersichtlichkeit nicht enthaltenen weiteren Unterelemente, sind:

- `<projectDesc>` ('project description'): beschreibt das Projekt, in dessen Rahmen die Quelle(n) digitalisiert und annotiert wurde(n).
- `<tagsDecl>` ('tagging declaration'): beschreibt die angewendete Tagging-Technik.

> - `<samplingDecl>` ('sampling declaration'): beschreibt die Gründe für die Textauswahl und die dabei angewandten Methoden. 

Reviewer 2: ```suggestion
- `<samplingDecl>` ('sampling declaration'): beschreibt die Gründe für die Textauswahl und die dabei angewandten Methoden.
- `<charDecl>` ('character declaration'): enthält Informationen über den Umgang mit im Quelltext vorkommenden Sonderzeichen, insbesondere Glyphen und sogenannte nicht-Standard-Zeichen, für die in den internationalen [Unicode-Zeichentabellen](https://www.unicode.org/charts/) kein Codepunkt existiert.
```
=> Hier ergänzt und Frage im Quiz unten umformuliert.

> - `<samplingDecl>` ('sampling declaration'): beschreibt die Gründe für die Textauswahl und die dabei angewandten Methoden. 

Reviewer 2: ```suggestion
- `<samplingDecl>` ('sampling declaration'): beschreibt die Gründe für die Textauswahl und die dabei angewandten Methoden.
- `<charDecl>` ('character declaration'): enthält Informationen über den Umgang mit im Quelltext vorkommenden Sonderzeichen, insbesondere Glyphen und sogenannte nicht-Standard-Zeichen, für die in den internationalen [Unicode-Zeichentabellen](https://www.unicode.org/charts/) kein Codepunkt existiert.
```
=> Hier ergänzt und Frage im Quiz unten umformuliert.

- **VIAF:** 'Virtual International Authority File' ist eine virtuelle internationale Normdatei, die nationale Normdateien zusammenführt und identische Normdatensätze verlinkt. VIAF-Identifikatoren kommen nicht nur bei der Katalogisierung von Bibliotheks- und Archivbeständen zum Einsatz, sondern werden auch von Museen, Projekten und Web-Anwendungen genutzt.

Schließlich enthält das Element `<availability>` Informationen zur Verfügbarkeit eines Textes, benennt Nutzungs- und Distributionseinschränkungen, gibt den Status des Urheberrechts und darüber hinaus die Lizenzierung an. Neben anderen im `<licence>`-Element zur Verfügung stehenden Attributen erlaubt z.B. das `@target`-Attribut, auf den Volltext der verwendeten Lizenz zu verweisen.

#### Quiz

<Quiz>
  <Quiz.Card>
    <Quiz.MultipleChoice>
      <Quiz.Question>
        Was wird im 'publication statement' dokumentiert?
      </Quiz.Question>

      <Quiz.MultipleChoice.Option>
        Das 'publication statement' enthält Publikationsangaben zur gedruckten Textvorlage.
      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>

>         Das 'publication statement' enthält alle Informationen rund um die Publikation und Distribution des elektronischen Texts.

Reviewer 2: ```suggestion
        Das 'publication statement' enthält alle Informationen rund um die Publikation und Distribution des elektronischen Textes.
```

Die **'revision description'** `<revisionDesc>` fasst die Revisionsgeschichte einer Datei zusammen und wird zumeist ans Ende des Headers gestellt. "Wer hat wann was geändert?" wird in `<change>` Statements wie folgt festgehalten:

```xml
<revisionDesc>
  <change when="2017-04-04" who="#em #mm" status="published">Freigegeben.</change>
  <change when="2017-04-03" who="#em" status="revised">Korrektur.</change>
  <change when="2017-04-01" who="#em" status="draft">Einleitung überarbeitet.</change>
  <change when="2017-03-28" who="#mm" status="draft">Erste Version fertig.</change>
</revisionDesc>
```

Die im `<change>`-Element angegebenen Attribute und ihre Werte dienen der genaueren Spezifizierung einer Änderung: `@when` enthält das Datum der Änderung, `@who` nennt die für die Änderung verantwortliche Person. Die Raute vor dem Bezeichner im Attributwert verweist auf die Definition dieser Person, die sich meist im 'statement of responsibility' des TEI-Header befindet und dort mit dem entsprechenden Bezeichner im `@xml:id`-Attribut gekennzeichnet ist:

```xml
<!-- ... -->

<persName xml:id="em">Erika Musterfrau</persName>
<persName xml:id="mm">Max Mustermann</persName>

<!-- ... -->
```

> Die Werte Attributs `@status` geben, wie der Name vermuten lässt, den Status des Dokuments nach der Änderung an, ob es sich z.B. um einen Entwurf, eine fertige Korrektur oder bereits für die Veröffentlichung freigegebene Version handelt. Die Reihenfolge der Änderungen sollte chronologisch, wahlweise auf- oder absteigend sortiert sein.

Reviewer 2: ```suggestion
Die Werte des Attributs `@status` geben, wie der Name vermuten lässt, den Status des Dokuments nach der Änderung an, ob es sich z.B. um einen Entwurf, eine fertige Korrektur oder bereits für die Veröffentlichung freigegebene Version handelt. Die Reihenfolge der Änderungen sollte chronologisch, wahlweise auf- oder absteigend sortiert sein.
```

      validate="input"
    >
      <Quiz.Question>
        Fügen Sie hier einen ersten Eintrag der Revisionsgeschichte mit dem Datum 21.04.2021 und dem Namenskürzel von Max Mustermann "mm" als den Urheber dieser Änderung hinzu.

>         Setzen Sie im `@status`-Attribut den Wert **`draft`** und als Beschreibungstext **"Erster Entwurf abgeschlossen."** ein

Reviewer 2: ```suggestion
        Setzen Sie im `@status`-Attribut den Wert **`draft`** und als Beschreibungstext **"Erster Entwurf abgeschlossen."** ein.
```

      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>
</Quiz>

### Inhalt & Kontext

Eine letzte der eingangs erwähnten fünf Hauptkomponenten des TEI-Header, die hier näher erläutert werden soll, ist die **'text-profile description'** `<profileDesc>`, die zum Einsatz kommt um nicht-bibliografische Aspekte von Texten wie z.B. die verwendeten Sprachen, den Produktionskontext, die daran Beteiligten und ihren Hintergrund zu beschreiben, oder im Text vorkommende Personen zu erfassen.

Im Element `<profileDesc>` stehen u.a. diese Unterelemente zur Verfügung:

- `<abstract>`: enthält eine Zusammenfassung.
- `<correspDesc>` ('correspondence description'): erfasst Korrespondenzbeschreibungen für Brief-Editionen.
- `<creation>`: erläutert die Textproduktion.
- `<langUsage>` ('language usage'): beschreibt Sprachen, Varietäten, Dialekte usw.

> - `<particDesc>` ('participation description'): beschreibt sowohl im Text als auch den Metadaten vorkommende Sprecher, "Stimmen" und Teilnehmer.

Reviewer 2: ```suggestion
- `<particDesc>` ('participation description'): beschreibt sowohl im Text als auch den Metadaten vorkommende Sprecher\*innen, "Stimmen" und Teilnehmer\*innen.
```

      solution={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt – ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n      </titleStmt>\n      <publicationStmt>\n        <p>Informationen zur Veröffentlichung</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Informationen über die Quelle</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text.</p>\n    </body>\n  </text>\n</TEI>"}
      validate="input"
    >
      <Quiz.Question>
        1. Geben Sie einen vollständigen Titel für das TEI-Dokument an. Ersetzen Sie das `<title>`-Element der Vorlage durch:

        - **Titel:** Die Filmwelt – ein Ausschnitt
        - **Untertitel:** Digitale Version
      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>

  <Quiz.Card>
    <Quiz.XmlCodeEditor

>       code={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt – ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n      </titleStmt>\n      <publicationStmt>\n        <p>Informationen zur Publikation</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Informationen über die Quelle</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text.</p>\n    </body>\n  </text>\n</TEI>"}

Reviewer 2: ```suggestion
      code={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt – ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n      </titleStmt>\n      <publicationStmt>\n        <p>Informationen zur Veröffentlichung</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Informationen über die Quelle</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text.</p>\n    </body>\n  </text>\n</TEI>"}
```
=> Habe das leider übersehen ... tut mir leid. Habe es auf "Veröffentlichung" geändert, weil es überall sonst auch so ist.

>       solution={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt – ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n        <respStmt>\n          <resp>TEI-Kodierung</resp>\n          <persName>Erika Musterfrau</persName>\n        </respStmt>\n      </titleStmt>\n      <publicationStmt>\n        <p>Informationen zur Publikation</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Informationen über die Quelle</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text.</p>\n    </body>\n  </text>\n</TEI>"}

Reviewer 2: ```suggestion
      solution={"<TEI xmlns=\"http://www.tei-c.org/ns/1.0\">\n  <teiHeader>\n    <fileDesc>\n      <titleStmt>\n        <title type=\"main\">Die Filmwelt – ein Ausschnitt</title>\n        <title type=\"sub\">Digitale Version</title>\n        <respStmt>\n          <resp>TEI-Kodierung</resp>\n          <persName>Erika Musterfrau</persName>\n        </respStmt>\n      </titleStmt>\n      <publicationStmt>\n        <p>Informationen zur Veröffentlichung</p>\n      </publicationStmt>\n      <sourceDesc>\n        <p>Informationen über die Quelle</p>\n      </sourceDesc>\n    </fileDesc>\n  </teiHeader>\n  <text>\n    <body>\n      <p>Text.</p>\n    </body>\n  </text>\n</TEI>"}
```
=> Habe das leider übersehen ... tut mir leid. Habe es auf "Veröffentlichung" geändert, weil es überall sonst auch so ist.

      validate="input"
    >
      <Quiz.Question>
        2. Ergänzen Sie bitte im **'title statement'** das **'statement of responsibility'** um folgende Elemente und Informationen:

        - **'responsibility':** TEI-Kodierung
        - **'personal name':** Erika Musterfrau als Urheber\*in der TEI-Kodierung
      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>
</Quiz>

### Quellenbeschreibung

> Die **'source description'** `<sourceDesc>` gibt Auskunft darüber, ob es sich um ein 'born-digital'-Dokument handelt oder ein Dokument, das auf anderen Quellen basiert – seien diese analog oder digital macht hier keinen Unterschied. Eine Beschreibung der Quellen kann als einfaches bibliografisches Zitat im Element **'bibliographic citation'** `<bibl>` erfolgen oder in vollständig strukturierter Form in **'structured bibliographic citation'** `<biblStruct>` vorgenommen werden. Für einen digital erstellten Text ohne physischen Quellen genügt eine kurze Angabe, die in einem `<p>`-Element ('paragraph') innerhalb der 'source description' einzufügen ist:

Reviewer 2: ```suggestion
Die **'source description'** `<sourceDesc>` gibt Auskunft darüber, ob es sich um ein 'born-digital'-Dokument handelt oder ein Dokument, das auf anderen Quellen basiert – seien diese analog oder digital macht hier keinen Unterschied. Eine Beschreibung der Quellen kann als einfaches bibliografisches Zitat im Element **'bibliographic citation'** `<bibl>` erfolgen oder in vollständig strukturierter Form in **'structured bibliographic citation'** `<biblStruct>` vorgenommen werden. Für einen digital erstellten Text ohne physische Quellen genügt eine kurze Angabe, die in einem `<p>`-Element ('paragraph') innerhalb der 'source description' einzufügen ist:
```

```xml
<sourceDesc>
  <p>Digitales Originaldokument.</p>
</sourceDesc>
```

Da es sich bei unserem Beispieltext _Die Filmwelt_ um einen digitalisierten Ausschnitt aus einer historischen Zeitschrift handelt, ist die physische Quelle zu nennen.

Eine minimale und unstrukturierte Form, ein bibliografisches Zitat zu annotieren, erfolgt im Element **'bibliographic citation'** `<bibl>`:

```xml
<bibl>Gold, Matthew K., ed. 2012. Debates in the Digital Humanities. Minneapolis, London: University of Minnesota Press.</bibl>
```

Darüber hinaus könnte man in diesem Literaturzitat **optional** einzelne Komponenten semantisch benennen, indem man innerhalb der Satzstruktur weitere Tags einfügt:

```xml
<bibl><editor>Matthew K. Gold</editor>, ed., <date>2012</date>. <title>Debates in the Digital Humanities.</title> <pubPlace>Minneapolis</pubPlace>, <pubPlace>London</pubPlace>: <publisher>University of Minnesota Press</publisher>.</bibl>
```

> Diese Auszeichnungsvariante ist nicht auf den TEI-Header beschränkt, sondern kann auch außerhalb verwendet werden. Sie eignet sich z.B. besonders für die Annotation von Bibliografien mit vielen Literaturzitaten, deren "Satzstruktur" der Textvorlage entsprechend abgebildet werden soll.

Reviewer 2: ```suggestion
Diese Auszeichnungsvariante ist nicht auf den TEI-Header beschränkt, sondern kann auch außerhalb verwendet werden. Sie eignet sich z.B. besonders für die Annotation von Bibliografien mit vielen Literaturzitaten, deren Satzstruktur der Textvorlage entsprechend abgebildet werden soll.
```
=> Nur Anführungszeichen entfernt.

      validate="input"
    >
      <Quiz.Question>
        **Bibliografische Informationen:** Die Filmwelt | Illustrierte Kino Revue | Herausgeber Felix Brasch | Zeitungsverlag Universale Ges. m. b. H. | Wien, VII., Hermanngasse 5 | Druckerei L. Beck und Sohn | Nr. 31, 1924 | Seite 11-12

        **Zu verwendende Elemente / Attribute / Werte:**

        1. Geben Sie Haupt- und Untertitel der Zeitschrift an `<title>` + `@type=main/sub`.
        2. Benennen Sie den Herausgeber der Zeitschrift `<editor>`.
        3. Ergänzen Sie im Impressum `<imprint>` den Verlag `<publisher>` mit allen zur Verfügung stehenden Angaben: neben dem Namen der Organisation `<orgName>` sollte auch die Adresse `<address>`, bestehend aus einer Adresszeile `<addrLine>`, enthalten sein.

>         4. Weisen Sie die Firma "L. Beck und Sohn" in einem 'statement of responsibility' mit den Elementen `<resp>` und `<orgName>` als Druckerei aus.

Reviewer 2: ```suggestion
        4. Weisen Sie die Firma "L. Beck und Sohn" in einem **'statement of responsibility'** mit den Elementen `<resp>` und `<orgName>` als Druckerei aus.
```
=> Nur fett ergänzt.

        5. Geben Sie "Wien" als Publikationsort und "1924" als Erscheinungsjahr mit den Elementen `<pubPlace>` und `<date>` an.
        6. Kodieren Sie, dass der gewählte Textausschnitt aus der Nr. "31, 1924" stammt und spezifizieren Sie den Ausschnitt Seite "11-12" in zwei `<biblScope>`-Elementen mit dem `@unit`-Attribut und dem Wert `volume` bzw. `page`.
      </Quiz.Question>
    </Quiz.XmlCodeEditor>
  </Quiz.Card>
</Quiz>

> Der Vollständigkeit halber sei an dieser Stelle noch erwähnt, dass für die bibliografische Strukturierung von Handschriften und frühen Drucken das Element **'manuscript description'** `<msDesc>` für die ausführliche Beschreibung im TEI-Header zur Verfügung steht (> Mehr Informationen hierzu in der [TEI-Dokumentation](https://tei-c.org/release/doc/tei-p5-doc/en/html/MS.html#msov)).

Reviewer 2: ```suggestion
Der Vollständigkeit halber sei an dieser Stelle noch erwähnt, dass für die bibliografische Strukturierung von Handschriften und frühen Drucken das Element **'manuscript description'** `<msDesc>` für die ausführliche Beschreibung im TEI-Header zur Verfügung steht (**\> Mehr Informationen hierzu in der [TEI-Dokumentation](https://tei-c.org/release/doc/tei-p5-doc/en/html/MS.html#msov))**.
```
=> Fett hat hier noch gefehlt.

- `<samplingDecl>` ('sampling declaration'): beschreibt die Gründe für die Textauswahl und die dabei angewandten Methoden.
- `<charDecl>` ('character declaration'): enthält Informationen über den Umgang mit im Quelltext vorkommenden Sonderzeichen, insbesondere Glyphen und sogenannte nicht-Standard-Zeichen, für die in den internationalen [Unicode-Zeichentabellen](https://www.unicode.org/charts/) kein Codepunkt existiert.

#### Quiz

<Quiz>
  <Quiz.Card>
    <Quiz.MultipleChoice>
      <Quiz.Question>
        Welche Aspekte eines Dokuments werden in der 'encoding description' des TEI-Header erfasst? (mehrere Antworten möglich)
      </Quiz.Question>

      <Quiz.MultipleChoice.Option isCorrect>
        Redaktionsprinzipien
      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>
        Informationen über Glyphen und sogenannte nicht-Standard-Zeichen.
      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>
        Projektbeschreibungen
      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>
        Tagverwendung
      </Quiz.MultipleChoice.Option>

      <Quiz.Message type="correct">

>         Alle Antworten sind in diesem Fall richtig.

Reviewer 2: => "Rückmeldung" formal anders als sonst.

      </Quiz.Message>
    </Quiz.MultipleChoice>
  </Quiz.Card>
</Quiz>

### Veröffentlichung

Nach der ausführlichen Beschreibung der verwendeten Quelle, müssten ebenso genaue Angaben zur Veröffentlichung des erstellten elektronischen Dokuments gemacht werden, wollte man den fertigen TEI-Beispieltext tatsächlich publizieren. Dazu wären in der bereits teilweise ausgefüllten 'file description' im **'publication statement'** `<publicationStmt>` Informationen zur Publikation und Verbreitung zu ergänzen. Neben dem Herausgeber, dem Erscheinungsort und -datum, könnten auch Angaben zur Verfügbarkeit gemacht werden, etwa unter welcher Lizenz das TEI-Dokument veröffentlicht wurde.

Es könnte sich z.B. um ein unveröffentlichtes Arbeitsdokument handeln, das gar nicht publiziert und nicht an Dritte weitergegeben wird und daher keine weiteren Metadaten diesbezüglich enthält ...

```xml
<publicationStmt>
  <p>Unveröffentlichtes Arbeitsdokument.</p>
</publicationStmt>
```

> ... oder es könnte sich um ein Dokument mit einer bestimmten Lizenzierung handeln wie z.B. "CC-BY 4.0" (Namensnennung 4.0 International). Sie erlaubt Dritten, das TEI-Dokument beliebig weiterzuverwenden, unter der Bedingung, bei erneuter Veröffentlichung die Urheber\*innen und Rechteinhaber\*innen des Ursprungsdokuments zu nennen. Eine andere Lizenz wäre z.B. "CC-BY-SA 4.0" (Namensnennung – Weitergabe unter gleichen Bedingungen 4.0 International). An diese Lizenz wäre neben der Namensnennung die Bedingung geknüpft, dass ein weiterverarbeitetes Dokument nur unter der gleichen Lizenz zu identen Bedingungen erneut publiziert werden dürfte. Die genannten und viele weitere Lizenzen für die Publikation digitaler Daten finden Sie auf den Webseiten der Organisation [Creative Commons](https://creativecommons.org/). Zu beachten ist jedenfalls, dass Sie ausschließlich Texte veröffentlichen und nur dann als Lizenzgeber*in auftreten sowie eine bestimmte Lizenz auswählen dürfen, wenn Sie dazu die Rechte besitzen.

Reviewer 2: ```suggestion
... oder es könnte sich um ein Dokument mit einer bestimmten Lizenzierung handeln wie z.B. "CC-BY 4.0" (Namensnennung 4.0 International). Sie erlaubt Dritten, das TEI-Dokument beliebig weiterzuverwenden, unter der Bedingung, bei erneuter Veröffentlichung die Urheber\*innen und Rechteinhaber\*innen des Ursprungsdokuments zu nennen. Eine andere Lizenz wäre z.B. "CC-BY-SA 4.0" (Namensnennung – Weitergabe unter gleichen Bedingungen 4.0 International). An diese Lizenz wäre neben der Namensnennung die Bedingung geknüpft, dass ein weiterverarbeitetes Dokument nur unter der gleichen Lizenz zu identen Bedingungen erneut publiziert werden dürfte. Die genannten und viele weitere Lizenzen für die Publikation digitaler Daten finden Sie auf den Webseiten der Organisation ['Creative Commons'](https://creativecommons.org/). Zu beachten ist jedenfalls, dass Sie ausschließlich Texte veröffentlichen und nur dann als Lizenzgeber*in auftreten sowie eine bestimmte Lizenz auswählen dürfen, wenn Sie dazu die Rechte besitzen.
```
=> Nur Anführungszeichen ergänzt.

```xml
<publicationStmt>
  <publisher>
    <orgName>Austrian Centre for Digital Humanities and Cultural Heritage</orgName>
    <email>acdh-ch@oeaw.ac.at</email>
    <idno type="VIAF">6515148451584915970000</idno>
  </publisher>
  <pubPlace>Wien</pubPlace>
  <date>2017</date>
  <availability>
    <licence target="https://creativecommons.org/licenses/by-sa/4.0/">
      <p>Veröffentlicht unter CC-BY-SA 4.0</p>
    </licence>
  </availability>
</publicationStmt>
```

Wie ersichtlich, würde ein einfacher Absatz ausreichen um Angaben über Veröffentlichungsbedingungen zu zu machen. Es empfiehlt sich jedoch auch hier, für ausführliche Informationen eine komplexere semantische Strukturierung vorzunehmen. Einige Elemente wie `<publisher>`, `<orgName>`, `<pubPlace>` oder `<date>` sind bereits aus der Quellenbeschreibung bekannt.

> Als weitere Elemente stehen in diesem Zusammenhang u.a. das `<email>`-Element **'electronic mail address'** für eine Emailadresse sowie das Element **'identifier'** `<idno>` zur Verfügung, das einen Bezeichner für die standardisierte Identifikation eines Objekts enthält. In diesem Fall ist das bezeichnete Objekt das Institut, das als 'publisher' auftritt. Das Attribut `@type` spezifiziert die Art der Identifikation und verweist auf den Eintrag für das **ACDH-CH** im [Virtual Authority File](https://viaf.org/viaf/6515148451584915970000/).

Reviewer 2: ```suggestion
Als weitere Elemente stehen in diesem Zusammenhang u.a. das `<email>`-Element **'electronic mail address'** für eine Emailadresse sowie das Element **'identifier'** `<idno>` zur Verfügung, das einen Bezeichner für die standardisierte Identifikation eines Objekts enthält. In diesem Fall ist das bezeichnete Objekt das Institut, das als 'publisher' auftritt. Das Attribut `@type` spezifiziert die Art der Identifikation und verweist auf den Eintrag für das **ACDH-CH** im ['Virtual Authority File'](https://viaf.org/viaf/6515148451584915970000/).
```
=> Nur Anführungszeichen ergänzt.

- **ISSN:** 'International Standard Serial Number' (Internationale Standardnummer für fortlaufende Sammelwerke) für die Identifikation von Zeitschriften und Schriftenreihen.
- **DOI:** 'Digital Object Identifier' für die Identifikation digitaler Objekte.
- **URI:** 'Uniform Resource Identifier' für Ressourcen im Internet. Das können einzelne Dokumente, aber auch Webseiten, Webservices und weiteres sein.
- **VIAF:** 'Virtual International Authority File' ist eine virtuelle internationale Normdatei, die nationale Normdateien zusammenführt und identische Normdatensätze verlinkt. VIAF-Identifikatoren kommen nicht nur bei der Katalogisierung von Bibliotheks- und Archivbeständen zum Einsatz, sondern werden auch von Museen, Projekten und Web-Anwendungen genutzt.

Schließlich enthält das Element `<availability>` Informationen zur Verfügbarkeit eines Textes, benennt Nutzungs- und Distributionseinschränkungen, gibt den Status des Urheberrechts und darüber hinaus die Lizenzierung an. Neben anderen im `<licence>`-Element zur Verfügung stehenden Attributen erlaubt z.B. das `@target`-Attribut, auf den Volltext der verwendeten Lizenz zu verweisen.

#### Quiz

> <Quiz>

Reviewer 2: => "Rückmeldung" nur automatisch - daher formal anders als sonst. Müsst Ihr entscheiden, ob das so passt. Vermutlich ist ab einem gewissen Fortschritt des Curriculums mehr auch nicht wirklich mehr notwendig.

Author: Hier habe ich eine Rückmeldung hinzugefügt.

      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>
        Nicht-bibliografische Aspekte.
      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>
        Inhaltliche Aspekte wie z.B. im Text vorkommende Personen, Sprachen usw.
      </Quiz.MultipleChoice.Option>

      <Quiz.MultipleChoice.Option isCorrect>
        Thematische Aspekte, im Sinne einer Verschlagwortung mit Hilfe eines standardisierten Klassifikationsschemas oder Thesaurus.
      </Quiz.MultipleChoice.Option>

      <Quiz.Message type="correct">
        **Richtig!**

        Die 'text-profile description' enthält keine bibliografischen Angaben. Jedoch können nicht-bibliografische und inhaltliche Aspekte sowie eine thematische Verschlagwortung enthalten sein.
      </Quiz.Message>

      <Quiz.Message type="incorrect">
        **Leider nicht ganz richtig!**

        Die 'text-profile description' enthält keine bibliografischen Angaben. Jedoch können nicht-bibliografische und inhaltliche Aspekte sowie eine thematische Verschlagwortung enthalten sein.
      </Quiz.Message>
    </Quiz.MultipleChoice>
  </Quiz.Card>
</Quiz>

## Weiterführende Information

> Das vollständige Modul zum TEI-Header auf den Webseiten der Text Encoding Initiative finden Sie [hier](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/HD.html).

Reviewer 2: ```suggestion
Das vollständige Modul zum TEI-Header auf den Webseiten der 'Text Encoding Initiative' finden Sie [hier](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/HD.html).
```
=> Nur Anführungszeichen ergänzt.
